
<!-- saved from url=(0057)http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-analysis.cgi -->
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US"><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252">

<script type="text/javascript">
 <!-- 
var stIsIE = /*@cc_on!@*/false;

sorttable = {
  init: function() {
    // quit if this function has already been called
    if (arguments.callee.done) return;
    // flag this function so we don't do the same thing twice
    arguments.callee.done = true;
    // kill the timer
    if (_timer) clearInterval(_timer);
    
    if (!document.createElement || !document.getElementsByTagName) return;
    
    sorttable.DATE_RE = /^(\d\d?)[\/\.-](\d\d?)[\/\.-]((\d\d)?\d\d)$/;
    
    forEach(document.getElementsByTagName('table'), function(table) {
      if (table.className.search(/\bsortable\b/) != -1) {
        sorttable.makeSortable(table);
      }
    });
    
  },
  
  makeSortable: function(table) {
    if (table.getElementsByTagName('thead').length == 0) {
      // table doesn't have a tHead. Since it should have, create one and
      // put the first table row in it.
      the = document.createElement('thead');
      the.appendChild(table.rows[0]);
      table.insertBefore(the,table.firstChild);
    }
    // Safari doesn't support table.tHead, sigh
    if (table.tHead == null) table.tHead = table.getElementsByTagName('thead')[0];
    
    if (table.tHead.rows.length != 1) return; // can't cope with two header rows
    
    // Sorttable v1 put rows with a class of "sortbottom" at the bottom (as
    // "total" rows, for example). This is B&R, since what you're supposed
    // to do is put them in a tfoot. So, if there are sortbottom rows,
    // for backwards compatibility, move them to tfoot (creating it if needed).
    sortbottomrows = [];
    for (var i=0; i<table.rows.length; i++) {
      if (table.rows[i].className.search(/\bsortbottom\b/) != -1) {
        sortbottomrows[sortbottomrows.length] = table.rows[i];
      }
    }
    if (sortbottomrows) {
      if (table.tFoot == null) {
        // table doesn't have a tfoot. Create one.
        tfo = document.createElement('tfoot');
        table.appendChild(tfo);
      }
      for (var i=0; i<sortbottomrows.length; i++) {
        tfo.appendChild(sortbottomrows[i]);
      }
      delete sortbottomrows;
    }
    
    // work through each column and calculate its type
    headrow = table.tHead.rows[0].cells;
    for (var i=0; i<headrow.length; i++) {
      // manually override the type with a sorttable_type attribute
      if (!headrow[i].className.match(/\bsorttable_nosort\b/)) { // skip this col
        mtch = headrow[i].className.match(/\bsorttable_([a-z0-9]+)\b/);
        if (mtch) { override = mtch[1]; }
	      if (mtch && typeof sorttable["sort_"+override] == 'function') {
	        headrow[i].sorttable_sortfunction = sorttable["sort_"+override];
	      } else {
	        headrow[i].sorttable_sortfunction = sorttable.guessType(table,i);
	      }
	      // make it clickable to sort
	      headrow[i].sorttable_columnindex = i;
	      headrow[i].sorttable_tbody = table.tBodies[0];
	      dean_addEvent(headrow[i],"click", function(e) {

          if (this.className.search(/\bsorttable_sorted\b/) != -1) {
            // if we're already sorted by this column, just 
            // reverse the table, which is quicker
            sorttable.reverse(this.sorttable_tbody);
            this.className = this.className.replace('sorttable_sorted',
                                                    'sorttable_sorted_reverse');
            this.removeChild(document.getElementById('sorttable_sortfwdind'));
            sortrevind = document.createElement('span');
            sortrevind.id = "sorttable_sortrevind";
            sortrevind.innerHTML = stIsIE ? '&nbsp<font face="webdings">5</font>' : '&nbsp;&#x25B4;';
            this.appendChild(sortrevind);
            return;
          }
          if (this.className.search(/\bsorttable_sorted_reverse\b/) != -1) {
            // if we're already sorted by this column in reverse, just 
            // re-reverse the table, which is quicker
            sorttable.reverse(this.sorttable_tbody);
            this.className = this.className.replace('sorttable_sorted_reverse',
                                                    'sorttable_sorted');
            this.removeChild(document.getElementById('sorttable_sortrevind'));
            sortfwdind = document.createElement('span');
            sortfwdind.id = "sorttable_sortfwdind";
            sortfwdind.innerHTML = stIsIE ? '&nbsp<font face="webdings">6</font>' : '&nbsp;&#x25BE;';
            this.appendChild(sortfwdind);
            return;
          }
          
          // remove sorttable_sorted classes
          theadrow = this.parentNode;
          forEach(theadrow.childNodes, function(cell) {
            if (cell.nodeType == 1) { // an element
              cell.className = cell.className.replace('sorttable_sorted_reverse','');
              cell.className = cell.className.replace('sorttable_sorted','');
            }
          });
          sortfwdind = document.getElementById('sorttable_sortfwdind');
          if (sortfwdind) { sortfwdind.parentNode.removeChild(sortfwdind); }
          sortrevind = document.getElementById('sorttable_sortrevind');
          if (sortrevind) { sortrevind.parentNode.removeChild(sortrevind); }
          
          this.className += ' sorttable_sorted';
          sortfwdind = document.createElement('span');
          sortfwdind.id = "sorttable_sortfwdind";
          sortfwdind.innerHTML = stIsIE ? '&nbsp<font face="webdings">6</font>' : '&nbsp;&#x25BE;';
          this.appendChild(sortfwdind);

	        // build an array to sort. This is a Schwartzian transform thing,
	        // i.e., we "decorate" each row with the actual sort key,
	        // sort based on the sort keys, and then put the rows back in order
	        // which is a lot faster because you only do getInnerText once per row
	        row_array = [];
	        col = this.sorttable_columnindex;
	        rows = this.sorttable_tbody.rows;
	        for (var j=0; j<rows.length; j++) {
	          row_array[row_array.length] = [sorttable.getInnerText(rows[j].cells[col]), rows[j]];
	        }
	        /* If you want a stable sort, uncomment the following line */
	        //sorttable.shaker_sort(row_array, this.sorttable_sortfunction);
	        /* and comment out this one */
	        row_array.sort(this.sorttable_sortfunction);
	        
	        tb = this.sorttable_tbody;
	        for (var j=0; j<row_array.length; j++) {
	          tb.appendChild(row_array[j][1]);
	        }
	        
	        delete row_array;
	      });
	    }
    }
  },
  
  guessType: function(table, column) {
    // guess the type of a column based on its first non-blank row
    sortfn = sorttable.sort_alpha;
    for (var i=0; i<table.tBodies[0].rows.length; i++) {
      text = sorttable.getInnerText(table.tBodies[0].rows[i].cells[column]);
      if (text != '') {
        if (text.match(/^-?[£$¤]?[\d,.]+%?$/)||text.match(/^\d+((\.|,)\d+)?(E|e)(\+|\-)\d+/)) {
          return sorttable.sort_numeric;
        }
        // check for a date: dd/mm/yyyy or dd/mm/yy 
        // can have / or . or - as separator
        // can be mm/dd as well
        possdate = text.match(sorttable.DATE_RE)
        if (possdate) {
          // looks like a date
          first = parseInt(possdate[1]);
          second = parseInt(possdate[2]);
          if (first > 12) {
            // definitely dd/mm
            return sorttable.sort_ddmm;
          } else if (second > 12) {
            return sorttable.sort_mmdd;
          } else {
            // looks like a date, but we can't tell which, so assume
            // that it's dd/mm (English imperialism!) and keep looking
            sortfn = sorttable.sort_ddmm;
          }
        }
      }
    }
    return sortfn;
  },
  
  getInnerText: function(node) {
    // gets the text we want to use for sorting for a cell.
    // strips leading and trailing whitespace.
    // this is *not* a generic getInnerText function; it's special to sorttable.
    // for example, you can override the cell text with a customkey attribute.
    // it also gets .value for <input> fields.
    
    hasInputs = (typeof node.getElementsByTagName == 'function') &&
                 node.getElementsByTagName('input').length;
    
    if (node.getAttribute("sorttable_customkey") != null) {
      return node.getAttribute("sorttable_customkey");
    }
    else if (typeof node.textContent != 'undefined' && !hasInputs) {
      return node.textContent.replace(/^\s+|\s+$/g, '');
    }
    else if (typeof node.innerText != 'undefined' && !hasInputs) {
      return node.innerText.replace(/^\s+|\s+$/g, '');
    }
    else if (typeof node.text != 'undefined' && !hasInputs) {
      return node.text.replace(/^\s+|\s+$/g, '');
    }
    else {
      switch (node.nodeType) {
        case 3:
          if (node.nodeName.toLowerCase() == 'input') {
            return node.value.replace(/^\s+|\s+$/g, '');
          }
        case 4:
          return node.nodeValue.replace(/^\s+|\s+$/g, '');
          break;
        case 1:
        case 11:
          var innerText = '';
          for (var i = 0; i < node.childNodes.length; i++) {
            innerText += sorttable.getInnerText(node.childNodes[i]);
          }
          return innerText.replace(/^\s+|\s+$/g, '');
          break;
        default:
          return '';
      }
    }
  },
  
  reverse: function(tbody) {
    // reverse the rows in a tbody
    newrows = [];
    for (var i=0; i<tbody.rows.length; i++) {
      newrows[newrows.length] = tbody.rows[i];
    }
    for (var i=newrows.length-1; i>=0; i--) {
       tbody.appendChild(newrows[i]);
    }
    delete newrows;
  },
  
  /* sort functions
     each sort function takes two parameters, a and b
     you are comparing a[0] and b[0] */
  /* Numeric order */
  sort_numeric: function(a,b) {
	aa = parseFloat(a[0].replace(/[^0-9.\-eE]/g,''));
	if (isNaN(aa)) aa = 0;
	bb = parseFloat(b[0].replace(/[^0-9.\-eE]/g,''));
	if (isNaN(bb)) bb = 0;
	return aa-bb;
    },
//   sort_numeric: function(a,b) {
//     aa = parseFloat(a[0].replace(/[^0-9.-eE]/g,''));
//     if (isNaN(aa)) aa = 0;
//     bb = parseFloat(b[0].replace(/[^0-9.-eE]/g,'')); 
//     if (isNaN(bb)) bb = 0;    
//     return aa-bb;
//   },

  /* alphabetical order */
  sort_alpha: function(a,b) {
    if (a[0]==b[0]) return 0;
    if (a[0]<b[0]) return -1;
    return 1;
  },

  /* sort dates formatted as ddmm (day first, month after) */
  sort_ddmm: function(a,b) {
    mtch = a[0].match(sorttable.DATE_RE);
    y = mtch[3]; m = mtch[2]; d = mtch[1];
    if (m.length == 1) m = '0'+m;
    if (d.length == 1) d = '0'+d;
    dt1 = y+m+d;
    mtch = b[0].match(sorttable.DATE_RE);
    y = mtch[3]; m = mtch[2]; d = mtch[1];
    if (m.length == 1) m = '0'+m;
    if (d.length == 1) d = '0'+d;
    dt2 = y+m+d;
    if (dt1==dt2) return 0;
    if (dt1<dt2) return -1;
    return 1;
  },

  /* sort dates formatted as mmdd (month first, day after) */
  sort_mmdd: function(a,b) {
    mtch = a[0].match(sorttable.DATE_RE);
    y = mtch[3]; d = mtch[2]; m = mtch[1];
    if (m.length == 1) m = '0'+m;
    if (d.length == 1) d = '0'+d;
    dt1 = y+m+d;
    mtch = b[0].match(sorttable.DATE_RE);
    y = mtch[3]; d = mtch[2]; m = mtch[1];
    if (m.length == 1) m = '0'+m;
    if (d.length == 1) d = '0'+d;
    dt2 = y+m+d;
    if (dt1==dt2) return 0;
    if (dt1<dt2) return -1;
    return 1;
  },
  
  shaker_sort: function(list, comp_func) {
    // A stable sort function to allow multi-level sorting of data
    // see: http://en.wikipedia.org/wiki/Cocktail_sort
    // thanks to Joseph Nahmias
    var b = 0;
    var t = list.length - 1;
    var swap = true;

    while(swap) {
        swap = false;
        for(var i = b; i < t; ++i) {
            if ( comp_func(list[i], list[i+1]) > 0 ) {
                var q = list[i]; list[i] = list[i+1]; list[i+1] = q;
                swap = true;
            }
        } // for
        t--;

        if (!swap) break;

        for(var i = t; i > b; --i) {
            if ( comp_func(list[i], list[i-1]) < 0 ) {
                var q = list[i]; list[i] = list[i-1]; list[i-1] = q;
                swap = true;
            }
        } // for
        b++;

    } // while(swap)
  }  
}

/* ******************************************************************
   Supporting functions: bundled here to avoid depending on a library
   ****************************************************************** */

// Dean Edwards/Matthias Miller/John Resig

/* for Mozilla/Opera9 */
if (document.addEventListener) {
    document.addEventListener("DOMContentLoaded", sorttable.init, false);
}

/* for Internet Explorer */
/*@cc_on @*/
/*@if (@_win32)
    document.write("<script id=__ie_onload defer src=javascript:void(0)><\/script>");
    var script = document.getElementById("__ie_onload");
    script.onreadystatechange = function() {
        if (this.readyState == "complete") {
            sorttable.init(); // call the onload handler
        }
    };
/*@end @*/

/* for Safari */
if (/WebKit/i.test(navigator.userAgent)) { // sniff
    var _timer = setInterval(function() {
        if (/loaded|complete/.test(document.readyState)) {
            sorttable.init(); // call the onload handler
        }
    }, 10);
}

/* for other browsers */
window.onload = sorttable.init;

// written by Dean Edwards, 2005
// with input from Tino Zijdel, Matthias Miller, Diego Perini

// http://dean.edwards.name/weblog/2005/10/add-event/

function dean_addEvent(element, type, handler) {
	if (element.addEventListener) {
		element.addEventListener(type, handler, false);
	} else {
		// assign each event handler a unique ID
		if (!handler.$$guid) handler.$$guid = dean_addEvent.guid++;
		// create a hash table of event types for the element
		if (!element.events) element.events = {};
		// create a hash table of event handlers for each element/event pair
		var handlers = element.events[type];
		if (!handlers) {
			handlers = element.events[type] = {};
			// store the existing event handler (if there is one)
			if (element["on" + type]) {
				handlers[0] = element["on" + type];
			}
		}
		// store the event handler in the hash table
		handlers[handler.$$guid] = handler;
		// assign a global event handler to do all the work
		element["on" + type] = handleEvent;
	}
};
// a counter used to create unique IDs
dean_addEvent.guid = 1;

function removeEvent(element, type, handler) {
	if (element.removeEventListener) {
		element.removeEventListener(type, handler, false);
	} else {
		// delete the event handler from the hash table
		if (element.events && element.events[type]) {
			delete element.events[type][handler.$$guid];
		}
	}
};

function handleEvent(event) {
	var returnValue = true;
	// grab the event object (IE uses a global event object)
	event = event || fixEvent(((this.ownerDocument || this.document || this).parentWindow || window).event);
	// get a reference to the hash table of event handlers
	var handlers = this.events[event.type];
	// execute each event handler
	for (var i in handlers) {
		this.$$handleEvent = handlers[i];
		if (this.$$handleEvent(event) === false) {
			returnValue = false;
		}
	}
	return returnValue;
};

function fixEvent(event) {
	// add W3C standard event methods
	event.preventDefault = fixEvent.preventDefault;
	event.stopPropagation = fixEvent.stopPropagation;
	return event;
};
fixEvent.preventDefault = function() {
	this.returnValue = false;
};
fixEvent.stopPropagation = function() {
  this.cancelBubble = true;
}

// Dean's forEach: http://dean.edwards.name/base/forEach.js
/*
	forEach, version 1.0
	Copyright 2006, Dean Edwards
	License: http://www.opensource.org/licenses/mit-license.php
*/

// array-like enumeration
if (!Array.forEach) { // mozilla already supports this
	Array.forEach = function(array, block, context) {
		for (var i = 0; i < array.length; i++) {
			block.call(context, array[i], i, array);
		}
	};
}

// generic enumeration
Function.prototype.forEach = function(object, block, context) {
	for (var key in object) {
		if (typeof this.prototype[key] == "undefined") {
			block.call(context, object[key], key, object);
		}
	}
};

// character enumeration
String.forEach = function(string, block, context) {
	Array.forEach(string.split(""), function(chr, index) {
		block.call(context, chr, index, string);
	});
};

// globally resolve forEach enumeration
var forEach = function(object, block, context) {
	if (object) {
		var resolve = Object; // default
		if (object instanceof Function) {
			// functions have a "length" property
			resolve = Function;
		} else if (object.forEach instanceof Function) {
			// the object implements a custom forEach method so use that
			object.forEach(block, context);
			return;
		} else if (typeof object == "string") {
			// the object is a string
			resolve = String;
		} else if (typeof object.length == "number") {
			// the object is array-like
			resolve = Array;
		}
		resolve.forEach(object, block, context);
	}
}

// --> 
</script>



<title>RSAT : oligo-analysis result</title>
<link rev="made" href="mailto:Jacques.van-Helden%5C%40univ-amu.fr">
<link rel="stylesheet" type="text/css" href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/main.css" media="screen,projection,print">
<link rel="stylesheet" type="text/css" href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/tabs.css" media="screen,projection,print">
<link rel="stylesheet" type="text/css" href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/chosen.css" media="screen,projection,print">
<link rel="stylesheet" type="text/css" href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/font-awesome.min.css" media="screen,projection,print">
<script src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/jquery.js.téléchargement" type="text/javascript"></script>
<script src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/RSAT_menu.js.téléchargement" type="text/javascript"></script>
<script src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/RSAT_tabs.js.téléchargement" type="text/javascript"></script>

</head>
<body class="results">
<style>#page-content-wrapper{
                    overflow: auto !important;
                    display: block;
                    height: 100%;
                }</style><script>
$(function(){
if(typeof demo !== "undefined"){
txt = demo.value;
if(txt){
document.getElementById("demo_descr").innerHTML = txt;
}

}
});
</script>        <link rel="stylesheet" href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/simple-sidebar.css">
        <link rel="stylesheet" type="text/css" href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/menu.css" media="screen,projection,print">
        <link rel="stylesheet" href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/colorbox.css">
	<link rel="stylesheet" type="text/css" href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/jquery-ui.css">
         <script src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/RSAT_menu.js.téléchargement" type="text/javascript"></script>
        <script src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/jquery.colorbox-min.js.téléchargement"></script>

        <script>
            $(document).ready(function(){
                $('.iframe').colorbox({iframe:true, innerWidth:'70%', innerHeight:'70%'});
		$("#menu-toggle").click(function(e){
			e.preventDefault();
			$("#wrapper").toggleClass('toggled');
		});
            });

        </script>

 

<div id="wrapper">
                   <!-- Sidebar -->
            <div id="sidebar-wrapper">
             <div id="menubody" style="padding-top:10px">

             <!--div id="tabmenu">
             <ul class="rsat">
                 <li><a class="active" href="index.php">RSAT</a></li>
                 <li><a href="NeAT_home.php" >NeAT</a></li>
                 <!--      <li><a href="http://www.rsat.eu/index_neat.html" target="_top">NeAT</a></li>>
             </ul>
             </div> < /#tabmenu -->
             <div id="content" class="rsat">

                 <div class="menu">
                     <h2>
                         <a href="http://pedagogix-tagc.univ-mrs.fr/rsat/index.php">
                             <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/RSAT_icon.jpg" style="max-width:150px;max-height:60px;" alt="RSAT server" border="0">
                                 </a>
                     </h2>
                 </div>

                 <!--      <p ><h2 style="border-style:solid;border-color:#cc6600;padding:0 10px;background-color:#F6E6CA;"><a href="http://www.eccb14.org/program/tutorials/cis-r" target="_blank">RSAT tutorial<br>at ECCB'14</a></h2></p>>

                  <p ><h2 style="border-style:solid;border-color:#cc6600;padding:0 10px;background-color:#F6E6CA;">
                  <a target='_blank' href="http://rsat.ulb.ac.be/eccb14/" >RSAT tutorial<br>at ECCB'14</a>
                  </h2></p-->
		<div style="padding:60px 0 0 10px;" align="center"><i class="fa fa-bar-chart  fa-lg"></i> <b>
4296</b> <i>organisms</i></div>
                <div>
			<input type="search" id="searchfunc" placeholder="Search" class="searchmenu" onkeypress="searchfunc()" onkeyup="searchfunc()">
		</div>
		<div align="right">
                     <b>New&nbsp;items</b><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" class="new">
                 </div>

                 <div class="menu_expand" onclick="expandAll(&#39;14&#39;)" id="expand"> &gt; view all tools</div>

                 <div class="menu">
                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;1&#39;)" id="heading1">Genomes and genes</div>
                     <div id="menu1" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/supported-organisms.cgi">supported organisms</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/gene-info_form.cgi"> gene information</a>
     

<a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/infer-operons_form.cgi"> infer operons</a><a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/get-orthologs_form.cgi">get orthologs</a>                         
                         
                         
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-genes_form.cgi">random gene selection</a>
                     </div>

                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;2&#39;)" id="heading2">Sequence tools <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu2" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-seq_form.cgi">retrieve sequence</a>
<a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-ensembl-seq_form.cgi">retrieve EnsEMBL seq</a><a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/fetch-sequences_form.php">fetch-sequences from UCSC</a>                         
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-seq-bed_form.cgi">sequences from bed/gff/vcf  <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <!--	  <a class="menu_item" href="http://www.rsat.eu/retrieve-ensembl-seq_form.cgi" >retrieve EnsEMBL seq</a>-->
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/purge-sequence_form.cgi">purge sequence</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-seq_form.cgi">convert sequences</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-seq_form.cgi">random sequences</a>
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;12&#39;)" id="heading12">Matrix tools <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu12" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-matrix_form.cgi">retrieve matrix <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-matrix_form.cgi">convert matrix</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/compare-matrices_form.cgi">compare matrices</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-clustering_form.cgi">matrix-clustering <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-distrib_form.cgi">matrix distrib</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-quality_form.cgi">matrix quality</a>
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;11&#39;)" id="heading11">Build control sets</div>
                     <div id="menu11" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-genes_form.cgi">random gene selection</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-seq_form.cgi">random sequences</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-genome-fragments_form.cgi">random genome fragments</a>
                         <!--	  <a class="menu_item" href="http://www.rsat.eu/random-genome-fragments_form.cgi" >random genome fragments</a>-->
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-motif_form.cgi">random-motif</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/permute-matrix_form.cgi">permute-matrix</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-sites_form.cgi">random-sites</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/implant-sites_form.cgi">implant-sites</a>
                     </div>
                 </div>

                 <div class="menu">
                     <div class="menu_heading_open" onclick="toggleMenu(&#39;3&#39;)" id="heading3">Motif discovery</div>
                     <div id="menu3" class="menu_collapsible" style="display: block;">
                         <a class="menu_separator">strings</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-analysis_form.cgi">oligo-analysis (words)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-diff_form.cgi">oligo-diff (words)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/dyad-analysis_form.cgi">dyad-analysis (spaced pairs)</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/pattern-assembly_form.cgi">pattern assembly</a>
                         <a class="menu_separator">strings with positional biais</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/position-analysis_form.cgi">position-analysis (words)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/local-word-analysis_form.cgi">local-word-analysis (word and spaced pairs)</a>
                         <a class="menu_separator">matrices</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/info-gibbs_form.cgi">info-gibbs</a>
                         <!--<a class="menu_item" href="consensus_form.cgi" >consensus</a>
                         	  <a class="menu_item" href="gibbs_form.cgi" >gibbs</a>-->
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;4&#39;)" id="heading4">Pattern matching <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu4" class="menu_collapsible">
                         <a class="menu_separator">matrices</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-scan_form.cgi">matrix-scan<br>(full options)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-scan-quick_form.cgi">matrix-scan (quick)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/crer-scan_form.cgi">crer-scan</a>
                         <!--<a class="menu_item" href="matrix-enrichment_form.cgi" >matrix-enrichment <img src="images/onebit_49.png" height="30" class="new"></img></a>-->
                         <!--	  <a class="menu_item" href="patser_form.cgi" >patser [discontinued]</a>-->
                         <!--	  <a class="menu_item" href="genome-scale-patser_form.cgi" >genome-scale patser [discontinued]</a>-->
                         <a class="menu_separator">strings</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/dna-pattern_form.cgi">dna-pattern</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/genome-scale-dna-pattern_form.cgi">genome-scale dna-pattern</a>
                     </div>
                 </div>

                 <div class="menu">
                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;10&#39;)" id="heading10">Comparative genomics&nbsp; <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu10" class="menu_collapsible">
<a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/get-orthologs_form.cgi">get orthologs</a>      
                     
                         <span class="menu_item" style="cursor:default;color:lightgray"> get orthologs-compara </span>                     
                         
                         
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/footprint-discovery_form.cgi">footprint-discovery</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/footprint-scan_form.cgi">footprint-scan <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new">
                         </a>   

                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;13&#39;)" id="heading13">NGS - ChIP-seq</div>
                     <div id="menu13" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/peak-motifs_form.cgi">peak-motifs</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/fetch-sequences_form.php">fetch-sequences from UCSC</a>                        <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-seq-bed_form.cgi">sequences from bed/gff/vcf  <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>

                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-genome-fragments_form.cgi">random genome fragments</a>
                         <!--	  <a class="menu_item" href="random-genome-fragments_form.cgi" >random genome fragments</a>-->
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;9&#39;)" id="heading9">Genetic variations (Var-tools) <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu9" class="menu_collapsible">
                     
                     <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/variation-info_form.cgi">Variation info <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-variation-seq_form.cgi">Retrieve variation sequences <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/variation-scan_form.cgi">Scan variations with motifs <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                                              
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-variations_form.cgi">Convert variation formats <img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                     </div>
                 </div>


                 <div class="menu">
                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;8&#39;)" id="heading8">Conversion/Utilities</div>
                     <div id="menu8" class="menu_collapsible">
                         <a class="menu_separator">Set comparisons / enrichment</a>
                         <!--<a class="menu_item_last" href="compare_classes_form.php?menu=RSAT" >compare classes/clusters</a> -->
                         <a class="menu_separator">Stats</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/classfreq_form.cgi">Frequency distribution</a>
                         <a class="menu_separator">sequences</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-seq_form.cgi">convert sequences</a>
                         <a class="menu_separator">matrices</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-matrix_form.cgi">convert matrix / logo</a>
                         <a class="menu_separator">background models</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/create-background-model_form.cgi">create background</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-background-model_form.cgi">convert background</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/seq-proba_form.cgi">sequence probability</a>
                         <a class="menu_separator">features</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-features_form.cgi">convert features</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/compare-features_form.cgi">compare features</a>
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;5&#39;)" id="heading5">Drawing</div>
                     <div id="menu5" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/feature-map_form.cgi">feature map</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/feature-map_form2.cgi">feature map 2</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/XYgraph_form.cgi">XY graph</a>
                     </div>


                 </div>

                 <div class="menu">
                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;6&#39;)" id="heading6">SOAP Web services</div>
                     <div id="menu6" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/htmllink.cgi?title=RSAT%20:%20Web%20services&amp;file=web_services.html">Programmatic interface to RSAT</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/htmllink.cgi?title=RSAT%20:%20Web%20services%20documentation&amp;file=web_services/RSATWS_documentation.xml">WSDL Documentation</a>
                         <!--a class="menu_item" href="web_services/RSATWS.wsdl" target=tools>WSDL</a>
                          <a class="menu_item" href="ws_clients.html" target=tools>Clients</a>
                          <a class="menu_item" href="ws_workflows.html" target=tools>Taverna workflows </a-->
                     </div>

                 </div>

                 <div class="menu">


                     <div class="menu_heading_open" onclick="toggleMenu(&#39;7&#39;)" id="heading7">Help &amp; Contact</div>
                         <div id="menu7" class="menu_collapsible_display">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/people.php">RSAT team</a>
                         <a class="menu_item" href="http://rsa-tools.github.io/teaching/index.html" target="_blank">Training material</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/tutorials.php">Tutorials</a>

                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/publications.cgi">Publications</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/credits.php">Credits</a>
                         <a class="menu_item" href="http://teaching.rsat.eu/download-request_form.cgi">Download</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/htmllink.cgi?title=RSAT-motif&amp;file=motif_databases/">Motif databases</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/htmllink.cgi?title=RSAT-Data&amp;file=data/">Data</a>

                        </div>

                         </div>

                        <a target="_top" href="http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-analysis.cgi#" style="font-size:3px;visibility:hidden">bug/dont understand</a>

                    </div>
               </div>

            </div><!-- /#sidebar-wrapper -->
<div class="container" id="page-content-wrapper"><span class="menu-toggle"><a href="http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-analysis.cgi#" id="menu-toggle"><i class="fa fa-bars fa-2x" style="color:green;"></i></a></span><h3 align="center"><a href="http://pedagogix-tagc.univ-mrs.fr/rsat/RSAT_home.cgi">RSAT</a> - oligo-analysis result</h3><blockquote class="information">
<font color="#006600"><b>Information: </b>Uploaded file	iHKG_TypeI_seq_723683692.fasta	78708 bytes</font>
</blockquote><br><hr size="3">
<pre>$RSAT/perl-scripts/convert-seq  -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm -from  fasta -to fasta  -mask non-dna -o $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta</pre>
<b>Server command</b><pre>$RSAT/perl-scripts/purge-sequence -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta -format fasta -o $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged -seqtype dna;$RSAT/perl-scripts/oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged -format fasta  -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible  -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 6 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_6nt.tab; $RSAT/perl-scripts/oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged -format fasta  -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible  -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 7 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_7nt.tab; $RSAT/perl-scripts/oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged -format fasta  -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible  -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 8 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_8nt.tab; cat $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_6nt.tab $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_7nt.tab $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_8nt.tab  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged -format fasta  -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible  -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01
</pre>
<blockquote class="information">
<font color="#006600"><b>Information: </b>One or several button(s) will appear at the bottom of this page, allowing you to send the result as input for a subsequent query.</font>
</blockquote><br><hr size="3">
<h2>Result</h2><pre> oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged -format fasta -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 6 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_6nt.tab
 Citation: van Helden et al. (1998). J Mol Biol 281(5), 827-42. 
 Program version              	1.169
 Quick counting mode          
 Detection of over-represented words (right-tail test)
 Oligomer length              	6
 Input file                   	$RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged
 Input format                 	fasta
 Output file                  	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_6nt.tab
 Discard overlapping matches
 Counted on both strands
 	grouped by pairs of reverse complements
 Background model             	upstream-noorf
 Organism                     	Candida_glabrata
 Background estimation method 	Frequency file
 Expected frequency file      	$RSAT/public_html/data/genomes/Candida_glabrata/oligo-frequencies/6nt_upstream-noorf_Candida_glabrata-noov-2str.freq
 Pseudo-frequency             	0.01
 Pseudo-frequency per oligo   	4.80769230769231e-06
 Sequence type                	DNA
 Nb of sequences              	73
 Sum of sequence lengths      	73000
 discarded residues           	NA (quick mode)	 (other letters than ACGT)
 discarded occurrences        	NA (quick mode)	 (contain discarded residues)
 nb possible positions        	NA (quick mode)
 total oligo occurrences      	71923
 total overlapping occurrences	1237
 total non overlapping occ    	70686
 alphabet size                	4
 nb possible oligomers        	2080
 oligomers tested for significance	2080
 Sequences:
	DIP5	1000
	CAGL0A02816g	1000
	COX12	1000
	HMX1	1000
	CAGL0A04499g	1000
	PHO84	1000
	CAGL0B03421g	1000
	CAGL0B04675g	1000
	CAGL0C03333g	1000
	ARG1	1000
	CAGL0D00198g	1000
	CAGL0D01958g	1000
	CAGL0D02640g	1000
	ACO1	1000
	CAGL0E01287g	1000
	CAGL0E03850g	1000
	SIT1	1000
	CAGL0E05588g	1000
	CAGL0E06116g	1000
	FET4	1000
	CAGL0F00231g	1000
	AUS1	1000
	TIR1	1000
	SCS7	1000
	CAGL0F04213g	1000
	CAGL0F04565g	1000
	CAGL0F05687g	1000
	CAGL0F05709g	1000
	CAGL0F05863g	1000
	FET3	1000
	UPC2B	1000
	AWP6	1000
	CAGL0H02491g	1000
	AFT1	1000
	CAGL0H03993g	1000
	CAGL0H09592g	1000
	CAGL0H09614g	1000
	CAGL0H09944g	1000
	CAGL0H09966g	1000
	CAGL0I00286g	1000
	TRR1	1000
	CAGL0I02508g	1000
	CAGL0I05060g	1000
	FTR1	1000
	CAGL0I07491g	1000
	CAGL0I07969g	1000
	PWP1	1000
	CAGL0J00451g	1000
	CAGL0J00847g	1000
	ALD5	1000
	CAGL0J03916g	1000
	CAGL0J04466g	1000
	CAGL0J05390g	1000
	LYS21	1000
	RTA1	1000
	CAGL0K01771g	1000
	CAGL0K02409g	1000
	CAGL0K02893g	1000
	CAGL0K03421g	1000
	TSA1	1000
	CAGL0K08338g	1000
	AP5	1000
	CAGL0K10824g	1000
	HIS1	1000
	CAGL0L01771g	1000
	MDH1	1000
	CAGL0L05258g	1000
	CAGL0L06776g	1000
	CAGL0L07502g	1000
	CYT1	1000
	CAGL0M04873g	1000
	CAGL0M05995g	1000
	CAGL0M08602g	1000

 column headers
	1	seq            	oligomer sequence
	2	id             	oligomer identifier
	3	exp_freq       	expected relative frequency
	4	occ            	observed occurrences
	5	exp_occ        	expected occurrences
	6	occ_P          	occurrence probability (binomial)
	7	occ_E          	E-value for occurrences (binomial)
	8	occ_sig        	occurrence significance (binomial)
	9	rank           	rank
	10	ovl_occ        	number of overlapping occurrences (discarded from the count)
	11	forbocc        	forbidden positions (to avoid self-overlap)
</pre>

<table class="sortable" cellspacing="1" cellpadding="3" border="0"><thead><tr valign="top">
<th align="left"><tt>seq</tt></th>
<th align="left"><tt>id</tt></th>
<th align="left"><tt>exp_freq</tt></th>
<th align="left"><tt>occ</tt></th>
<th align="left"><tt>exp_occ</tt></th>
<th align="left"><tt>occ_P</tt></th>
<th align="left"><tt>occ_E</tt></th>
<th align="left"><tt>occ_sig</tt></th>
<th align="left"><tt>rank</tt></th>
<th align="left"><tt>ovl_occ</tt></th>
<th align="left"><tt>forbocc</tt></th>
</tr></thead>
<tbody>

<tr valign="top">
<td><tt>gcaccc</tt></td>
<td><tt>gcaccc|gggtgc</tt></td>
<td><tt>0.0001475937898</tt></td>
<td><tt>31</tt></td>
<td><tt>10.62</tt></td>
<td><tt>2.8e-07</tt></td>
<td><tt>5.9e-04</tt></td>
<td><tt>3.23</tt></td>
<td><tt>1</tt></td>
<td><tt>0</tt></td>
<td><tt>155</tt></td>
</tr>

<tr valign="top">
<td><tt>ggtgca</tt></td>
<td><tt>ggtgca|tgcacc</tt></td>
<td><tt>0.0002393173094</tt></td>
<td><tt>41</tt></td>
<td><tt>17.21</tt></td>
<td><tt>7.8e-07</tt></td>
<td><tt>1.6e-03</tt></td>
<td><tt>2.79</tt></td>
<td><tt>2</tt></td>
<td><tt>1</tt></td>
<td><tt>205</tt></td>
</tr>

<tr valign="top">
<td><tt>acaagg</tt></td>
<td><tt>acaagg|ccttgt</tt></td>
<td><tt>0.0004401004091</tt></td>
<td><tt>57</tt></td>
<td><tt>31.65</tt></td>
<td><tt>3.2e-05</tt></td>
<td><tt>6.6e-02</tt></td>
<td><tt>1.18</tt></td>
<td><tt>3</tt></td>
<td><tt>0</tt></td>
<td><tt>285</tt></td>
</tr>

<tr valign="top">
<td><tt>ccttcc</tt></td>
<td><tt>ccttcc|ggaagg</tt></td>
<td><tt>0.0004917933892</tt></td>
<td><tt>60</tt></td>
<td><tt>35.37</tt></td>
<td><tt>0.00010</tt></td>
<td><tt>2.1e-01</tt></td>
<td><tt>0.68</tt></td>
<td><tt>4</tt></td>
<td><tt>0</tt></td>
<td><tt>300</tt></td>
</tr>

<tr valign="top">
<td><tt>gacggc</tt></td>
<td><tt>gacggc|gccgtc</tt></td>
<td><tt>0.0000754127383</tt></td>
<td><tt>16</tt></td>
<td><tt>5.42</tt></td>
<td><tt>0.00017</tt></td>
<td><tt>3.6e-01</tt></td>
<td><tt>0.45</tt></td>
<td><tt>5</tt></td>
<td><tt>0</tt></td>
<td><tt>80</tt></td>
</tr>

<tr valign="top">
<td><tt>agaagg</tt></td>
<td><tt>agaagg|ccttct</tt></td>
<td><tt>0.0005945489472</tt></td>
<td><tt>67</tt></td>
<td><tt>42.76</tt></td>
<td><tt>0.00036</tt></td>
<td><tt>7.6e-01</tt></td>
<td><tt>0.12</tt></td>
<td><tt>6</tt></td>
<td><tt>0</tt></td>
<td><tt>335</tt></td>
</tr>

<tr valign="top">
<td><tt>agttcc</tt></td>
<td><tt>agttcc|ggaact</tt></td>
<td><tt>0.0004104715058</tt></td>
<td><tt>50</tt></td>
<td><tt>29.52</tt></td>
<td><tt>0.00037</tt></td>
<td><tt>7.6e-01</tt></td>
<td><tt>0.12</tt></td>
<td><tt>7</tt></td>
<td><tt>0</tt></td>
<td><tt>250</tt></td>
</tr>

</tbody><tfoot></tfoot></table><pre> Host name	pedagogix
 Job started	2020-01-31.095153
 Job done	2020-01-31.095154
 Seconds	1.18
	user	1.18
	system	0.04
	cuser	0.43
	csystem	0.03
 oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged -format fasta -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 7 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_7nt.tab
 Citation: van Helden et al. (1998). J Mol Biol 281(5), 827-42. 
 Program version              	1.169
 Quick counting mode          
 Detection of over-represented words (right-tail test)
 Oligomer length              	7
 Input file                   	$RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged
 Input format                 	fasta
 Output file                  	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_7nt.tab
 Discard overlapping matches
 Counted on both strands
 	grouped by pairs of reverse complements
 Background model             	upstream-noorf
 Organism                     	Candida_glabrata
 Background estimation method 	Frequency file
 Expected frequency file      	$RSAT/public_html/data/genomes/Candida_glabrata/oligo-frequencies/7nt_upstream-noorf_Candida_glabrata-noov-2str.freq
 Pseudo-frequency             	0.01
 Pseudo-frequency per oligo   	1.220703125e-06
 Sequence type                	DNA
 Nb of sequences              	73
 Sum of sequence lengths      	73000
 discarded residues           	NA (quick mode)	 (other letters than ACGT)
 discarded occurrences        	NA (quick mode)	 (contain discarded residues)
 nb possible positions        	NA (quick mode)
 total oligo occurrences      	71781
 total overlapping occurrences	1011
 total non overlapping occ    	70770
 alphabet size                	4
 nb possible oligomers        	8192
 oligomers tested for significance	8192
 Sequences:
	DIP5	1000
	CAGL0A02816g	1000
	COX12	1000
	HMX1	1000
	CAGL0A04499g	1000
	PHO84	1000
	CAGL0B03421g	1000
	CAGL0B04675g	1000
	CAGL0C03333g	1000
	ARG1	1000
	CAGL0D00198g	1000
	CAGL0D01958g	1000
	CAGL0D02640g	1000
	ACO1	1000
	CAGL0E01287g	1000
	CAGL0E03850g	1000
	SIT1	1000
	CAGL0E05588g	1000
	CAGL0E06116g	1000
	FET4	1000
	CAGL0F00231g	1000
	AUS1	1000
	TIR1	1000
	SCS7	1000
	CAGL0F04213g	1000
	CAGL0F04565g	1000
	CAGL0F05687g	1000
	CAGL0F05709g	1000
	CAGL0F05863g	1000
	FET3	1000
	UPC2B	1000
	AWP6	1000
	CAGL0H02491g	1000
	AFT1	1000
	CAGL0H03993g	1000
	CAGL0H09592g	1000
	CAGL0H09614g	1000
	CAGL0H09944g	1000
	CAGL0H09966g	1000
	CAGL0I00286g	1000
	TRR1	1000
	CAGL0I02508g	1000
	CAGL0I05060g	1000
	FTR1	1000
	CAGL0I07491g	1000
	CAGL0I07969g	1000
	PWP1	1000
	CAGL0J00451g	1000
	CAGL0J00847g	1000
	ALD5	1000
	CAGL0J03916g	1000
	CAGL0J04466g	1000
	CAGL0J05390g	1000
	LYS21	1000
	RTA1	1000
	CAGL0K01771g	1000
	CAGL0K02409g	1000
	CAGL0K02893g	1000
	CAGL0K03421g	1000
	TSA1	1000
	CAGL0K08338g	1000
	AP5	1000
	CAGL0K10824g	1000
	HIS1	1000
	CAGL0L01771g	1000
	MDH1	1000
	CAGL0L05258g	1000
	CAGL0L06776g	1000
	CAGL0L07502g	1000
	CYT1	1000
	CAGL0M04873g	1000
	CAGL0M05995g	1000
	CAGL0M08602g	1000

 column headers
	1	seq            	oligomer sequence
	2	id             	oligomer identifier
	3	exp_freq       	expected relative frequency
	4	occ            	observed occurrences
	5	exp_occ        	expected occurrences
	6	occ_P          	occurrence probability (binomial)
	7	occ_E          	E-value for occurrences (binomial)
	8	occ_sig        	occurrence significance (binomial)
	9	rank           	rank
	10	ovl_occ        	number of overlapping occurrences (discarded from the count)
	11	forbocc        	forbidden positions (to avoid self-overlap)
</pre>

<table class="sortable" cellspacing="1" cellpadding="3" border="0"><thead><tr valign="top">
<th align="left"><tt>seq</tt></th>
<th align="left"><tt>id</tt></th>
<th align="left"><tt>exp_freq</tt></th>
<th align="left"><tt>occ</tt></th>
<th align="left"><tt>exp_occ</tt></th>
<th align="left"><tt>occ_P</tt></th>
<th align="left"><tt>occ_E</tt></th>
<th align="left"><tt>occ_sig</tt></th>
<th align="left"><tt>rank</tt></th>
<th align="left"><tt>ovl_occ</tt></th>
<th align="left"><tt>forbocc</tt></th>
</tr></thead>
<tbody>

<tr valign="top">
<td><tt>gggtgca</tt></td>
<td><tt>gggtgca|tgcaccc</tt></td>
<td><tt>0.0000419830038</tt></td>
<td><tt>19</tt></td>
<td><tt>3.01</tt></td>
<td><tt>6e-10</tt></td>
<td><tt>4.9e-06</tt></td>
<td><tt>5.31</tt></td>
<td><tt>1</tt></td>
<td><tt>0</tt></td>
<td><tt>114</tt></td>
</tr>

<tr valign="top">
<td><tt>cgtagac</tt></td>
<td><tt>cgtagac|gtctacg</tt></td>
<td><tt>0.0000337673463</tt></td>
<td><tt>11</tt></td>
<td><tt>2.42</tt></td>
<td><tt>4.7e-05</tt></td>
<td><tt>3.8e-01</tt></td>
<td><tt>0.42</tt></td>
<td><tt>2</tt></td>
<td><tt>0</tt></td>
<td><tt>66</tt></td>
</tr>

<tr valign="top">
<td><tt>ggtgcaa</tt></td>
<td><tt>ggtgcaa|ttgcacc</tt></td>
<td><tt>0.0000789534625</tt></td>
<td><tt>17</tt></td>
<td><tt>5.67</tt></td>
<td><tt>9e-05</tt></td>
<td><tt>7.4e-01</tt></td>
<td><tt>0.13</tt></td>
<td><tt>3</tt></td>
<td><tt>0</tt></td>
<td><tt>102</tt></td>
</tr>

<tr valign="top">
<td><tt>ctacgac</tt></td>
<td><tt>ctacgac|gtcgtag</tt></td>
<td><tt>0.0000309234648</tt></td>
<td><tt>10</tt></td>
<td><tt>2.22</tt></td>
<td><tt>0.00011</tt></td>
<td><tt>8.9e-01</tt></td>
<td><tt>0.05</tt></td>
<td><tt>4</tt></td>
<td><tt>0</tt></td>
<td><tt>60</tt></td>
</tr>

</tbody><tfoot></tfoot></table><pre> Host name	pedagogix
 Job started	2020-01-31.095155
 Job done	2020-01-31.095158
 Seconds	2.53
	user	2.53
	system	0.05
	cuser	0.48
	csystem	0.04
 oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged -format fasta -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 8 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_8nt.tab
 Citation: van Helden et al. (1998). J Mol Biol 281(5), 827-42. 
 Program version              	1.169
 Quick counting mode          
 Detection of over-represented words (right-tail test)
 Oligomer length              	8
 Input file                   	$RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged
 Input format                 	fasta
 Output file                  	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_8nt.tab
 Discard overlapping matches
 Counted on both strands
 	grouped by pairs of reverse complements
 Background model             	upstream-noorf
 Organism                     	Candida_glabrata
 Background estimation method 	Frequency file
 Expected frequency file      	$RSAT/public_html/data/genomes/Candida_glabrata/oligo-frequencies/8nt_upstream-noorf_Candida_glabrata-noov-2str.freq
 Pseudo-frequency             	0.01
 Pseudo-frequency per oligo   	3.03988326848249e-07
 Sequence type                	DNA
 Nb of sequences              	73
 Sum of sequence lengths      	73000
 discarded residues           	NA (quick mode)	 (other letters than ACGT)
 discarded occurrences        	NA (quick mode)	 (contain discarded residues)
 nb possible positions        	NA (quick mode)
 total oligo occurrences      	71724
 total overlapping occurrences	502
 total non overlapping occ    	71222
 alphabet size                	4
 nb possible oligomers        	32896
 oligomers tested for significance	32887
 Sequences:
	DIP5	1000
	CAGL0A02816g	1000
	COX12	1000
	HMX1	1000
	CAGL0A04499g	1000
	PHO84	1000
	CAGL0B03421g	1000
	CAGL0B04675g	1000
	CAGL0C03333g	1000
	ARG1	1000
	CAGL0D00198g	1000
	CAGL0D01958g	1000
	CAGL0D02640g	1000
	ACO1	1000
	CAGL0E01287g	1000
	CAGL0E03850g	1000
	SIT1	1000
	CAGL0E05588g	1000
	CAGL0E06116g	1000
	FET4	1000
	CAGL0F00231g	1000
	AUS1	1000
	TIR1	1000
	SCS7	1000
	CAGL0F04213g	1000
	CAGL0F04565g	1000
	CAGL0F05687g	1000
	CAGL0F05709g	1000
	CAGL0F05863g	1000
	FET3	1000
	UPC2B	1000
	AWP6	1000
	CAGL0H02491g	1000
	AFT1	1000
	CAGL0H03993g	1000
	CAGL0H09592g	1000
	CAGL0H09614g	1000
	CAGL0H09944g	1000
	CAGL0H09966g	1000
	CAGL0I00286g	1000
	TRR1	1000
	CAGL0I02508g	1000
	CAGL0I05060g	1000
	FTR1	1000
	CAGL0I07491g	1000
	CAGL0I07969g	1000
	PWP1	1000
	CAGL0J00451g	1000
	CAGL0J00847g	1000
	ALD5	1000
	CAGL0J03916g	1000
	CAGL0J04466g	1000
	CAGL0J05390g	1000
	LYS21	1000
	RTA1	1000
	CAGL0K01771g	1000
	CAGL0K02409g	1000
	CAGL0K02893g	1000
	CAGL0K03421g	1000
	TSA1	1000
	CAGL0K08338g	1000
	AP5	1000
	CAGL0K10824g	1000
	HIS1	1000
	CAGL0L01771g	1000
	MDH1	1000
	CAGL0L05258g	1000
	CAGL0L06776g	1000
	CAGL0L07502g	1000
	CYT1	1000
	CAGL0M04873g	1000
	CAGL0M05995g	1000
	CAGL0M08602g	1000

 column headers
	1	seq            	oligomer sequence
	2	id             	oligomer identifier
	3	exp_freq       	expected relative frequency
	4	occ            	observed occurrences
	5	exp_occ        	expected occurrences
	6	occ_P          	occurrence probability (binomial)
	7	occ_E          	E-value for occurrences (binomial)
	8	occ_sig        	occurrence significance (binomial)
	9	rank           	rank
	10	ovl_occ        	number of overlapping occurrences (discarded from the count)
	11	forbocc        	forbidden positions (to avoid self-overlap)
</pre>

<table class="sortable" cellspacing="1" cellpadding="3" border="0"><thead><tr valign="top">
<th align="left"><tt>seq</tt></th>
<th align="left"><tt>id</tt></th>
<th align="left"><tt>exp_freq</tt></th>
<th align="left"><tt>occ</tt></th>
<th align="left"><tt>exp_occ</tt></th>
<th align="left"><tt>occ_P</tt></th>
<th align="left"><tt>occ_E</tt></th>
<th align="left"><tt>occ_sig</tt></th>
<th align="left"><tt>rank</tt></th>
<th align="left"><tt>ovl_occ</tt></th>
<th align="left"><tt>forbocc</tt></th>
</tr></thead>
<tbody>

<tr valign="top">
<td><tt>aacaatgg</tt></td>
<td><tt>aacaatgg|ccattgtt</tt></td>
<td><tt>0.0001246748140</tt></td>
<td><tt>24</tt></td>
<td><tt>8.94</tt></td>
<td><tt>2.2e-05</tt></td>
<td><tt>7.3e-01</tt></td>
<td><tt>0.14</tt></td>
<td><tt>1</tt></td>
<td><tt>0</tt></td>
<td><tt>168</tt></td>
</tr>

<tr valign="top">
<td><tt>aaagaaaa</tt></td>
<td><tt>aaagaaaa|ttttcttt</tt></td>
<td><tt>0.0004164557330</tt></td>
<td><tt>55</tt></td>
<td><tt>29.87</tt></td>
<td><tt>2.4e-05</tt></td>
<td><tt>7.9e-01</tt></td>
<td><tt>0.10</tt></td>
<td><tt>2</tt></td>
<td><tt>4</tt></td>
<td><tt>385</tt></td>
</tr>

<tr valign="top">
<td><tt>gagtttcc</tt></td>
<td><tt>gagtttcc|ggaaactc</tt></td>
<td><tt>0.0000262541351</tt></td>
<td><tt>10</tt></td>
<td><tt>1.88</tt></td>
<td><tt>2.8e-05</tt></td>
<td><tt>9.3e-01</tt></td>
<td><tt>0.03</tt></td>
<td><tt>3</tt></td>
<td><tt>0</tt></td>
<td><tt>70</tt></td>
</tr>

</tbody><tfoot></tfoot></table><pre> Host name	pedagogix
 Job started	2020-01-31.095158
 Job done	2020-01-31.095204
 Seconds	5.96
	user	5.96
	system	0.15
	cuser	0.26
;	csystem	0.04
</pre>
<h2>Pattern assembly</h2>
<pre>pattern-assembly command: $RSAT/perl-scripts/pattern-assembly -v 1 -subst 1 -toppat 50 -2str -max_asmb_nb 10 -i $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.tab -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.asmb<p>
</p></pre><pre>; pattern-assembly  -v 1 -subst 1 -toppat 50 -2str -max_asmb_nb 10 -i $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.tab -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.asmb
; Input file	/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.tab
; Output file	/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.asmb
; Input score column           	8
; Output score column          	0
; Strand-insensitive assembly
; Max flanking bases           	1
; Max substitutions            	1
; Min matches                  	0
; Min weight                   	0
; Max assembly size            	50
; Max assembly width           	0
; Top number of patterns       	50
; Number of input patterns     	14
;

; assembly # 1	seed: gggtgca	4 assembled patterns	length 8
; alignt	 rev_cpl	score
gggtgca.	.tgcaccc	5.31
gggtgc..	..gcaccc	3.23
.ggtgca.	.tgcacc.	2.79
.ggtgcaa	ttgcacc.	0.13
gggtgcaa	ttgcaccc	5.31	best consensus

; assembly # 2	seed: acaagg	1 assembled patterns	length 6
;alignt	rev_cpl	score
acaagg	ccttgt	1.18
acaagg	ccttgt	1.18	best consensus

; assembly # 3	seed: ccttcc	1 assembled patterns	length 6
;alignt	rev_cpl	score
ccttcc	ggaagg	0.68
ccttcc	ggaagg	0.68	best consensus

; assembly # 4	seed: gacggc	1 assembled patterns	length 6
;alignt	rev_cpl	score
gacggc	gccgtc	0.45
gacggc	gccgtc	0.45	best consensus

; assembly # 5	seed: cgtagac	1 assembled patterns	length 7
;alignt	rev_cpl	score
cgtagac	gtctacg	0.42
cgtagac	gtctacg	0.42	best consensus

; assembly # 6	seed: aacaatgg	1 assembled patterns	length 8
; alignt	 rev_cpl	score
aacaatgg	ccattgtt	0.14
aacaatgg	ccattgtt	0.14	best consensus

; assembly # 7	seed: agttcc	1 assembled patterns	length 6
;alignt	rev_cpl	score
agttcc	ggaact	0.12
agttcc	ggaact	0.12	best consensus

; assembly # 8	seed: agaagg	1 assembled patterns	length 6
;alignt	rev_cpl	score
agaagg	ccttct	0.12
agaagg	ccttct	0.12	best consensus

; assembly # 9	seed: aaagaaaa	1 assembled patterns	length 8
; alignt	 rev_cpl	score
aaagaaaa	ttttcttt	0.10
aaagaaaa	ttttcttt	0.10	best consensus

; assembly # 10	seed: ctacgac	1 assembled patterns	length 7
;alignt	rev_cpl	score
ctacgac	gtcgtag	0.05
ctacgac	gtcgtag	0.05	best consensus
; Maximal number of assemblies is reached (10). Remaining patterns: 0.
; Host name	pedagogix
; Job started	2020-01-31.095204
; Job done	2020-01-31.095206
; Seconds	1.73
;	user	1.73
;	system	0.04
;	cuser	0
;	csystem	0
</pre>
<hr><pre><b>Command to generate matrices (PSSM):</b> $RSAT/perl-scripts/matrix-from-patterns -v 1 -logo  -seq $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta -format fasta -asmb $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.asmb -min_weight 5 -flanks 2 -max_asmb_nb 10 -uth Pval 0.00025 -bginput -markov 0 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm
</pre>
<hr>
<pre><b>Matrix conversion command:</b> $RSAT/perl-scripts/convert-matrix -v 1  -i $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt -from tab -to tab  -return counts -return consensus -return logo -logo_format png  -logo_opt '-e'  -logo_opt '-M'  -logo_dir $RSAT/public_html/tmp/www-data/2020/01/31/logos -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_2020-01-31.095226_wD7cN4_converted.tab
</pre><h2>Matrices</h2><pre>; convert-matrix  -v 1 -i $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt -from tab -to tab -return counts -return consensus -return logo -logo_format png -logo_opt -e -logo_opt -M -logo_dir $RSAT/public_html/tmp/www-data/2020/01/31/logos -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_2020-01-31.095226_wD7cN4_converted.tab
; Input files
;	input	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt
; Input format        	tab
; Output files
;	output	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_2020-01-31.095226_wD7cN4_converted.tab
; Output format       	tab
; pseudo-weight       	1
a	49	49	8	10	5	5	4	2	127	47	37	38
c	28	24	6	3	7	2	5	124	4	29	32	29
g	36	29	115	115	120	5	126	8	3	29	20	33
t	26	37	10	11	7	127	4	5	5	34	50	39
//
; consensus	rwGGGTGCAaww
; consensus.rc	WWTTGCACCCWY
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_1_lo.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_1_lo.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(1).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(1).png" height="120"></a>
;
a	11	20	57	0	57	57	0	0	19	22
c	14	11	0	57	0	0	0	0	9	5
g	16	6	0	0	0	0	57	57	14	11
t	16	20	0	0	0	0	0	0	15	19
//
; consensus	kwACAAGGww
; consensus.rc	WWCCTTGTWM
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_2_lo.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_2_lo.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(2).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(2).png" height="120"></a>
;
a	8	10	0	0	0	0	0	0	22	16
c	22	23	60	60	0	0	60	60	20	23
g	8	4	0	0	0	0	0	0	3	6
t	22	23	0	0	60	60	0	0	15	15
//
; consensus	yyCCTTCChh
; consensus.rc	DDGGAAGGRR
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_3_lo.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_3_lo.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(3).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(3).png" height="120"></a>
;
a	7	6	0	16	0	0	0	0	8	7
c	4	3	0	0	16	0	0	16	2	2
g	3	4	16	0	0	16	16	0	4	5
t	2	3	0	0	0	0	0	0	2	2
//
; consensus	mrGACGGCrr
; consensus.rc	YYGCCGTCYK
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_4_lo.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_4_lo.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(4).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(4).png" height="120"></a>
;
a	37	42	7	5	4	96	6	102	3	49	36
c	21	21	92	2	5	0	7	1	93	20	22
g	23	15	1	94	0	6	90	4	7	13	21
t	27	30	8	7	99	6	5	1	5	26	29
//
; consensus	wwCGTAGACaw
; consensus.rc	WTGTCTACGWW
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_5_lo.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_5_lo.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(5).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(5).png" height="120"></a>
;
a	34	35	96	100	3	99	95	5	7	3	34	40
c	20	21	2	2	99	2	3	1	1	3	28	13
g	17	34	4	4	1	5	1	7	95	97	26	27
t	36	17	5	1	4	1	8	94	4	4	19	27
//
; consensus	wrAACAATGGmd
; consensus.rc	HKCCATTGTTYW
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_6_lo.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_6_lo.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(6).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(6).png" height="120"></a>
;
a	9	15	50	0	0	0	0	0	16	10
c	15	15	0	0	0	0	50	50	9	14
g	13	11	0	50	0	0	0	0	8	14
t	13	9	0	0	50	50	0	0	17	12
//
; consensus	bmAGTTCCws
; consensus.rc	SWGGAACTKV
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_7_lo.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_7_lo.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(7).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(7).png" height="120"></a>
;
a	23	20	67	0	67	67	0	0	15	18
c	8	18	0	0	0	0	0	0	16	16
g	16	17	0	67	0	0	67	67	22	24
t	20	12	0	0	0	0	0	0	14	9
//
; consensus	wvAGAAGGgr
; consensus.rc	YCCCTTCTBW
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_8_lo.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_8_lo.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(8).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(8).png" height="120"></a>
;
a	251	273	523	530	524	175	525	524	517	512	282	255
c	87	95	16	8	3	23	11	9	11	13	69	77
g	119	122	8	14	17	330	17	19	14	27	92	93
t	104	71	14	9	17	33	8	9	19	9	118	136
//
; consensus	aaAAArAAAAaa
; consensus.rc	TTTTTTYTTTTT
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_9_lo.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_9_lo.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(9).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(9).png" height="120"></a>
;
a	23	28	4	2	85	13	12	88	4	42	32
c	16	13	84	0	3	73	3	1	79	14	22
g	25	17	2	5	2	3	73	0	7	9	15
t	29	35	3	86	3	4	5	4	3	28	24
//
; consensus	kwCTACGACww
; consensus.rc	WWGTCGTAGWM
<a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_10_l.png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt_10_l.png" height="120"></a><a href="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(10).png"><img src="./iHKG_TypeI_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tx(10).png" height="120"></a>
;
; Host name	pedagogix
; Job started	2020-01-31.095227
; Job done	2020-01-31.095235
; Seconds	0.64
;	user	0.64
;	system	0.14
;	cuser	7.41
;	csystem	0.76
</pre><h2>Result file(s)</h2><table class="resultlink">
<tbody><tr><th>Content</th><th>URL</th></tr><tr><td>Input sequence</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta</a></td></tr><tr><td>Purged sequence</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta.purged</a></td></tr><tr><td>6-mers</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_6nt.tab">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_6nt.tab</a></td></tr><tr><td>7-mers</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_7nt.tab">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_7nt.tab</a></td></tr><tr><td>8-mers</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_8nt.tab">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_8nt.tab</a></td></tr><tr><td>merged oligos</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.tab">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.tab</a></td></tr><tr><td>Assembly</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.asmb">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.asmb</a></td></tr><tr><td>Significance matrices</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_sig_matrices.tf">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_sig_matrices.tf</a></td></tr><tr><td>Rescaled significance matrices</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_sig_matrices_rescaled.tf">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_sig_matrices_rescaled.tf</a></td></tr><tr><td>Final matrices (transfac format)</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tf">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tf</a></td></tr><tr><td>Final matrices (tab format)</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.txt</a></td></tr></tbody></table><hr size="3"><table class="nextstep" cellspacing="0" cellpadding="10" border="0" nowrap="">
<tbody><tr>

</tr><tr valign="top" align="center">
    <th valign="BOTTOM" align="CENTER" colspan="6">
	Next step
    </th>

</tr>

<tr><td valign="bottom" align="center">
<form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/dna-pattern_form.cgi">
<input type="hidden" name="title" value="">
<input type="hidden" name="pattern_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.tab">
<input type="hidden" name="sequence_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta">
<input type="hidden" name="sequence_format" value="fasta">
<input type="submit" value="string-based pattern matching (dna-pattern)">
</form>
</td>

<td valign="bottom" align="center">
<form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/pattern-assembly_form.cgi">
<input type="hidden" name="local_pattern_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.tab">
<input type="hidden" name="subst" value="1">
<input type="hidden" name="maxfl" value="1">
<input type="hidden" name="sc" value="auto">
<input type="hidden" name="strand" value="insensitive">
<input type="submit" value="pattern assembly">
</form>
</td>
</tr>

<tr><td valign="bottom" align="center"><form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-scan_form.cgi"><input type="hidden" name="matrix_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tf"><input type="hidden" name="matrix_format" value="transfac"><input type="hidden" name="sequence_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.095151_oySyXm.fasta"><input type="hidden" name="sequence_format" value="fasta"><input type="submit" value="matrix-based pattern matching (matrix-scan)"></form></td></tr><tr><td valign="bottom" align="center"><form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-matrix_form.cgi"><input type="hidden" name="matrix_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW_pssm_count_matrices.tf"><input type="hidden" name="matrix_format" value="transfac"><input type="submit" value="matrix conversion"></form></td></tr><tr></tr>

<tr>
<td valign="bottom" align="center">
<form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/XYgraph_form.cgi">
<input type="hidden" name="title1" value="oligo-analysis result">
<input type="hidden" name="title2" value="">
<input type="hidden" name="XYgraph_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.tab">
<input type="hidden" name="xcol" value="5">
<input type="hidden" name="xleg1" value="expected occurrences">
<input type="hidden" name="ycol" value="4">
<input type="hidden" name="yleg1" value="observed occurrences">
<input type="submit" value="XY graph">
</form>
</td>

  <td>
<form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/classfreq_form.cgi">
<input type="hidden" name="transferred_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.095151_6rhkbW.tab">
<input type="submit" value="Frequency distribution">
 </form>
  </td>
</tr>


</tbody></table>
<hr size="3">

</div></div><div id="cboxOverlay" style="display: none;"></div><div id="colorbox" class="" role="dialog" tabindex="-1" style="display: none;"><div id="cboxWrapper"><div><div id="cboxTopLeft" style="float: left;"></div><div id="cboxTopCenter" style="float: left;"></div><div id="cboxTopRight" style="float: left;"></div></div><div style="clear: left;"><div id="cboxMiddleLeft" style="float: left;"></div><div id="cboxContent" style="float: left;"><div id="cboxTitle" style="float: left;"></div><div id="cboxCurrent" style="float: left;"></div><button type="button" id="cboxPrevious"></button><button type="button" id="cboxNext"></button><button type="button" id="cboxSlideshow"></button><div id="cboxLoadingOverlay" style="float: left;"></div><div id="cboxLoadingGraphic" style="float: left;"></div></div><div id="cboxMiddleRight" style="float: left;"></div></div><div style="clear: left;"><div id="cboxBottomLeft" style="float: left;"></div><div id="cboxBottomCenter" style="float: left;"></div><div id="cboxBottomRight" style="float: left;"></div></div></div><div style="position: absolute; width: 9999px; visibility: hidden; display: none; max-width: none;"></div></div></body></html>