
<!-- saved from url=(0057)http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-analysis.cgi -->
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US"><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252">

<script type="text/javascript">
 <!-- 
var stIsIE = /*@cc_on!@*/false;

sorttable = {
  init: function() {
    // quit if this function has already been called
    if (arguments.callee.done) return;
    // flag this function so we don't do the same thing twice
    arguments.callee.done = true;
    // kill the timer
    if (_timer) clearInterval(_timer);
    
    if (!document.createElement || !document.getElementsByTagName) return;
    
    sorttable.DATE_RE = /^(\d\d?)[\/\.-](\d\d?)[\/\.-]((\d\d)?\d\d)$/;
    
    forEach(document.getElementsByTagName('table'), function(table) {
      if (table.className.search(/\bsortable\b/) != -1) {
        sorttable.makeSortable(table);
      }
    });
    
  },
  
  makeSortable: function(table) {
    if (table.getElementsByTagName('thead').length == 0) {
      // table doesn't have a tHead. Since it should have, create one and
      // put the first table row in it.
      the = document.createElement('thead');
      the.appendChild(table.rows[0]);
      table.insertBefore(the,table.firstChild);
    }
    // Safari doesn't support table.tHead, sigh
    if (table.tHead == null) table.tHead = table.getElementsByTagName('thead')[0];
    
    if (table.tHead.rows.length != 1) return; // can't cope with two header rows
    
    // Sorttable v1 put rows with a class of "sortbottom" at the bottom (as
    // "total" rows, for example). This is B&R, since what you're supposed
    // to do is put them in a tfoot. So, if there are sortbottom rows,
    // for backwards compatibility, move them to tfoot (creating it if needed).
    sortbottomrows = [];
    for (var i=0; i<table.rows.length; i++) {
      if (table.rows[i].className.search(/\bsortbottom\b/) != -1) {
        sortbottomrows[sortbottomrows.length] = table.rows[i];
      }
    }
    if (sortbottomrows) {
      if (table.tFoot == null) {
        // table doesn't have a tfoot. Create one.
        tfo = document.createElement('tfoot');
        table.appendChild(tfo);
      }
      for (var i=0; i<sortbottomrows.length; i++) {
        tfo.appendChild(sortbottomrows[i]);
      }
      delete sortbottomrows;
    }
    
    // work through each column and calculate its type
    headrow = table.tHead.rows[0].cells;
    for (var i=0; i<headrow.length; i++) {
      // manually override the type with a sorttable_type attribute
      if (!headrow[i].className.match(/\bsorttable_nosort\b/)) { // skip this col
        mtch = headrow[i].className.match(/\bsorttable_([a-z0-9]+)\b/);
        if (mtch) { override = mtch[1]; }
	      if (mtch && typeof sorttable["sort_"+override] == 'function') {
	        headrow[i].sorttable_sortfunction = sorttable["sort_"+override];
	      } else {
	        headrow[i].sorttable_sortfunction = sorttable.guessType(table,i);
	      }
	      // make it clickable to sort
	      headrow[i].sorttable_columnindex = i;
	      headrow[i].sorttable_tbody = table.tBodies[0];
	      dean_addEvent(headrow[i],"click", function(e) {

          if (this.className.search(/\bsorttable_sorted\b/) != -1) {
            // if we're already sorted by this column, just 
            // reverse the table, which is quicker
            sorttable.reverse(this.sorttable_tbody);
            this.className = this.className.replace('sorttable_sorted',
                                                    'sorttable_sorted_reverse');
            this.removeChild(document.getElementById('sorttable_sortfwdind'));
            sortrevind = document.createElement('span');
            sortrevind.id = "sorttable_sortrevind";
            sortrevind.innerHTML = stIsIE ? '&nbsp<font face="webdings">5</font>' : '&nbsp;&#x25B4;';
            this.appendChild(sortrevind);
            return;
          }
          if (this.className.search(/\bsorttable_sorted_reverse\b/) != -1) {
            // if we're already sorted by this column in reverse, just 
            // re-reverse the table, which is quicker
            sorttable.reverse(this.sorttable_tbody);
            this.className = this.className.replace('sorttable_sorted_reverse',
                                                    'sorttable_sorted');
            this.removeChild(document.getElementById('sorttable_sortrevind'));
            sortfwdind = document.createElement('span');
            sortfwdind.id = "sorttable_sortfwdind";
            sortfwdind.innerHTML = stIsIE ? '&nbsp<font face="webdings">6</font>' : '&nbsp;&#x25BE;';
            this.appendChild(sortfwdind);
            return;
          }
          
          // remove sorttable_sorted classes
          theadrow = this.parentNode;
          forEach(theadrow.childNodes, function(cell) {
            if (cell.nodeType == 1) { // an element
              cell.className = cell.className.replace('sorttable_sorted_reverse','');
              cell.className = cell.className.replace('sorttable_sorted','');
            }
          });
          sortfwdind = document.getElementById('sorttable_sortfwdind');
          if (sortfwdind) { sortfwdind.parentNode.removeChild(sortfwdind); }
          sortrevind = document.getElementById('sorttable_sortrevind');
          if (sortrevind) { sortrevind.parentNode.removeChild(sortrevind); }
          
          this.className += ' sorttable_sorted';
          sortfwdind = document.createElement('span');
          sortfwdind.id = "sorttable_sortfwdind";
          sortfwdind.innerHTML = stIsIE ? '&nbsp<font face="webdings">6</font>' : '&nbsp;&#x25BE;';
          this.appendChild(sortfwdind);

	        // build an array to sort. This is a Schwartzian transform thing,
	        // i.e., we "decorate" each row with the actual sort key,
	        // sort based on the sort keys, and then put the rows back in order
	        // which is a lot faster because you only do getInnerText once per row
	        row_array = [];
	        col = this.sorttable_columnindex;
	        rows = this.sorttable_tbody.rows;
	        for (var j=0; j<rows.length; j++) {
	          row_array[row_array.length] = [sorttable.getInnerText(rows[j].cells[col]), rows[j]];
	        }
	        /* If you want a stable sort, uncomment the following line */
	        //sorttable.shaker_sort(row_array, this.sorttable_sortfunction);
	        /* and comment out this one */
	        row_array.sort(this.sorttable_sortfunction);
	        
	        tb = this.sorttable_tbody;
	        for (var j=0; j<row_array.length; j++) {
	          tb.appendChild(row_array[j][1]);
	        }
	        
	        delete row_array;
	      });
	    }
    }
  },
  
  guessType: function(table, column) {
    // guess the type of a column based on its first non-blank row
    sortfn = sorttable.sort_alpha;
    for (var i=0; i<table.tBodies[0].rows.length; i++) {
      text = sorttable.getInnerText(table.tBodies[0].rows[i].cells[column]);
      if (text != '') {
        if (text.match(/^-?[£$¤]?[\d,.]+%?$/)||text.match(/^\d+((\.|,)\d+)?(E|e)(\+|\-)\d+/)) {
          return sorttable.sort_numeric;
        }
        // check for a date: dd/mm/yyyy or dd/mm/yy 
        // can have / or . or - as separator
        // can be mm/dd as well
        possdate = text.match(sorttable.DATE_RE)
        if (possdate) {
          // looks like a date
          first = parseInt(possdate[1]);
          second = parseInt(possdate[2]);
          if (first > 12) {
            // definitely dd/mm
            return sorttable.sort_ddmm;
          } else if (second > 12) {
            return sorttable.sort_mmdd;
          } else {
            // looks like a date, but we can't tell which, so assume
            // that it's dd/mm (English imperialism!) and keep looking
            sortfn = sorttable.sort_ddmm;
          }
        }
      }
    }
    return sortfn;
  },
  
  getInnerText: function(node) {
    // gets the text we want to use for sorting for a cell.
    // strips leading and trailing whitespace.
    // this is *not* a generic getInnerText function; it's special to sorttable.
    // for example, you can override the cell text with a customkey attribute.
    // it also gets .value for <input> fields.
    
    hasInputs = (typeof node.getElementsByTagName == 'function') &&
                 node.getElementsByTagName('input').length;
    
    if (node.getAttribute("sorttable_customkey") != null) {
      return node.getAttribute("sorttable_customkey");
    }
    else if (typeof node.textContent != 'undefined' && !hasInputs) {
      return node.textContent.replace(/^\s+|\s+$/g, '');
    }
    else if (typeof node.innerText != 'undefined' && !hasInputs) {
      return node.innerText.replace(/^\s+|\s+$/g, '');
    }
    else if (typeof node.text != 'undefined' && !hasInputs) {
      return node.text.replace(/^\s+|\s+$/g, '');
    }
    else {
      switch (node.nodeType) {
        case 3:
          if (node.nodeName.toLowerCase() == 'input') {
            return node.value.replace(/^\s+|\s+$/g, '');
          }
        case 4:
          return node.nodeValue.replace(/^\s+|\s+$/g, '');
          break;
        case 1:
        case 11:
          var innerText = '';
          for (var i = 0; i < node.childNodes.length; i++) {
            innerText += sorttable.getInnerText(node.childNodes[i]);
          }
          return innerText.replace(/^\s+|\s+$/g, '');
          break;
        default:
          return '';
      }
    }
  },
  
  reverse: function(tbody) {
    // reverse the rows in a tbody
    newrows = [];
    for (var i=0; i<tbody.rows.length; i++) {
      newrows[newrows.length] = tbody.rows[i];
    }
    for (var i=newrows.length-1; i>=0; i--) {
       tbody.appendChild(newrows[i]);
    }
    delete newrows;
  },
  
  /* sort functions
     each sort function takes two parameters, a and b
     you are comparing a[0] and b[0] */
  /* Numeric order */
  sort_numeric: function(a,b) {
	aa = parseFloat(a[0].replace(/[^0-9.\-eE]/g,''));
	if (isNaN(aa)) aa = 0;
	bb = parseFloat(b[0].replace(/[^0-9.\-eE]/g,''));
	if (isNaN(bb)) bb = 0;
	return aa-bb;
    },
//   sort_numeric: function(a,b) {
//     aa = parseFloat(a[0].replace(/[^0-9.-eE]/g,''));
//     if (isNaN(aa)) aa = 0;
//     bb = parseFloat(b[0].replace(/[^0-9.-eE]/g,'')); 
//     if (isNaN(bb)) bb = 0;    
//     return aa-bb;
//   },

  /* alphabetical order */
  sort_alpha: function(a,b) {
    if (a[0]==b[0]) return 0;
    if (a[0]<b[0]) return -1;
    return 1;
  },

  /* sort dates formatted as ddmm (day first, month after) */
  sort_ddmm: function(a,b) {
    mtch = a[0].match(sorttable.DATE_RE);
    y = mtch[3]; m = mtch[2]; d = mtch[1];
    if (m.length == 1) m = '0'+m;
    if (d.length == 1) d = '0'+d;
    dt1 = y+m+d;
    mtch = b[0].match(sorttable.DATE_RE);
    y = mtch[3]; m = mtch[2]; d = mtch[1];
    if (m.length == 1) m = '0'+m;
    if (d.length == 1) d = '0'+d;
    dt2 = y+m+d;
    if (dt1==dt2) return 0;
    if (dt1<dt2) return -1;
    return 1;
  },

  /* sort dates formatted as mmdd (month first, day after) */
  sort_mmdd: function(a,b) {
    mtch = a[0].match(sorttable.DATE_RE);
    y = mtch[3]; d = mtch[2]; m = mtch[1];
    if (m.length == 1) m = '0'+m;
    if (d.length == 1) d = '0'+d;
    dt1 = y+m+d;
    mtch = b[0].match(sorttable.DATE_RE);
    y = mtch[3]; d = mtch[2]; m = mtch[1];
    if (m.length == 1) m = '0'+m;
    if (d.length == 1) d = '0'+d;
    dt2 = y+m+d;
    if (dt1==dt2) return 0;
    if (dt1<dt2) return -1;
    return 1;
  },
  
  shaker_sort: function(list, comp_func) {
    // A stable sort function to allow multi-level sorting of data
    // see: http://en.wikipedia.org/wiki/Cocktail_sort
    // thanks to Joseph Nahmias
    var b = 0;
    var t = list.length - 1;
    var swap = true;

    while(swap) {
        swap = false;
        for(var i = b; i < t; ++i) {
            if ( comp_func(list[i], list[i+1]) > 0 ) {
                var q = list[i]; list[i] = list[i+1]; list[i+1] = q;
                swap = true;
            }
        } // for
        t--;

        if (!swap) break;

        for(var i = t; i > b; --i) {
            if ( comp_func(list[i], list[i-1]) < 0 ) {
                var q = list[i]; list[i] = list[i-1]; list[i-1] = q;
                swap = true;
            }
        } // for
        b++;

    } // while(swap)
  }  
}

/* ******************************************************************
   Supporting functions: bundled here to avoid depending on a library
   ****************************************************************** */

// Dean Edwards/Matthias Miller/John Resig

/* for Mozilla/Opera9 */
if (document.addEventListener) {
    document.addEventListener("DOMContentLoaded", sorttable.init, false);
}

/* for Internet Explorer */
/*@cc_on @*/
/*@if (@_win32)
    document.write("<script id=__ie_onload defer src=javascript:void(0)><\/script>");
    var script = document.getElementById("__ie_onload");
    script.onreadystatechange = function() {
        if (this.readyState == "complete") {
            sorttable.init(); // call the onload handler
        }
    };
/*@end @*/

/* for Safari */
if (/WebKit/i.test(navigator.userAgent)) { // sniff
    var _timer = setInterval(function() {
        if (/loaded|complete/.test(document.readyState)) {
            sorttable.init(); // call the onload handler
        }
    }, 10);
}

/* for other browsers */
window.onload = sorttable.init;

// written by Dean Edwards, 2005
// with input from Tino Zijdel, Matthias Miller, Diego Perini

// http://dean.edwards.name/weblog/2005/10/add-event/

function dean_addEvent(element, type, handler) {
	if (element.addEventListener) {
		element.addEventListener(type, handler, false);
	} else {
		// assign each event handler a unique ID
		if (!handler.$$guid) handler.$$guid = dean_addEvent.guid++;
		// create a hash table of event types for the element
		if (!element.events) element.events = {};
		// create a hash table of event handlers for each element/event pair
		var handlers = element.events[type];
		if (!handlers) {
			handlers = element.events[type] = {};
			// store the existing event handler (if there is one)
			if (element["on" + type]) {
				handlers[0] = element["on" + type];
			}
		}
		// store the event handler in the hash table
		handlers[handler.$$guid] = handler;
		// assign a global event handler to do all the work
		element["on" + type] = handleEvent;
	}
};
// a counter used to create unique IDs
dean_addEvent.guid = 1;

function removeEvent(element, type, handler) {
	if (element.removeEventListener) {
		element.removeEventListener(type, handler, false);
	} else {
		// delete the event handler from the hash table
		if (element.events && element.events[type]) {
			delete element.events[type][handler.$$guid];
		}
	}
};

function handleEvent(event) {
	var returnValue = true;
	// grab the event object (IE uses a global event object)
	event = event || fixEvent(((this.ownerDocument || this.document || this).parentWindow || window).event);
	// get a reference to the hash table of event handlers
	var handlers = this.events[event.type];
	// execute each event handler
	for (var i in handlers) {
		this.$$handleEvent = handlers[i];
		if (this.$$handleEvent(event) === false) {
			returnValue = false;
		}
	}
	return returnValue;
};

function fixEvent(event) {
	// add W3C standard event methods
	event.preventDefault = fixEvent.preventDefault;
	event.stopPropagation = fixEvent.stopPropagation;
	return event;
};
fixEvent.preventDefault = function() {
	this.returnValue = false;
};
fixEvent.stopPropagation = function() {
  this.cancelBubble = true;
}

// Dean's forEach: http://dean.edwards.name/base/forEach.js
/*
	forEach, version 1.0
	Copyright 2006, Dean Edwards
	License: http://www.opensource.org/licenses/mit-license.php
*/

// array-like enumeration
if (!Array.forEach) { // mozilla already supports this
	Array.forEach = function(array, block, context) {
		for (var i = 0; i < array.length; i++) {
			block.call(context, array[i], i, array);
		}
	};
}

// generic enumeration
Function.prototype.forEach = function(object, block, context) {
	for (var key in object) {
		if (typeof this.prototype[key] == "undefined") {
			block.call(context, object[key], key, object);
		}
	}
};

// character enumeration
String.forEach = function(string, block, context) {
	Array.forEach(string.split(""), function(chr, index) {
		block.call(context, chr, index, string);
	});
};

// globally resolve forEach enumeration
var forEach = function(object, block, context) {
	if (object) {
		var resolve = Object; // default
		if (object instanceof Function) {
			// functions have a "length" property
			resolve = Function;
		} else if (object.forEach instanceof Function) {
			// the object implements a custom forEach method so use that
			object.forEach(block, context);
			return;
		} else if (typeof object == "string") {
			// the object is a string
			resolve = String;
		} else if (typeof object.length == "number") {
			// the object is array-like
			resolve = Array;
		}
		resolve.forEach(object, block, context);
	}
}

// --> 
</script>



<title>RSAT : oligo-analysis result</title>
<link rev="made" href="mailto:Jacques.van-Helden%5C%40univ-amu.fr">
<link rel="stylesheet" type="text/css" href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/main.css" media="screen,projection,print">
<link rel="stylesheet" type="text/css" href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/tabs.css" media="screen,projection,print">
<link rel="stylesheet" type="text/css" href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/chosen.css" media="screen,projection,print">
<link rel="stylesheet" type="text/css" href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/font-awesome.min.css" media="screen,projection,print">
<script src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/jquery.js.téléchargement" type="text/javascript"></script>
<script src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/RSAT_menu.js.téléchargement" type="text/javascript"></script>
<script src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/RSAT_tabs.js.téléchargement" type="text/javascript"></script>

</head>
<body class="results">
<style>#page-content-wrapper{
                    overflow: auto !important;
                    display: block;
                    height: 100%;
                }</style><script>
$(function(){
if(typeof demo !== "undefined"){
txt = demo.value;
if(txt){
document.getElementById("demo_descr").innerHTML = txt;
}

}
});
</script>        <link rel="stylesheet" href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/simple-sidebar.css">
        <link rel="stylesheet" type="text/css" href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/menu.css" media="screen,projection,print">
        <link rel="stylesheet" href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/colorbox.css">
	<link rel="stylesheet" type="text/css" href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/jquery-ui.css">
         <script src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/RSAT_menu.js.téléchargement" type="text/javascript"></script>
        <script src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/jquery.colorbox-min.js.téléchargement"></script>

        <script>
            $(document).ready(function(){
                $('.iframe').colorbox({iframe:true, innerWidth:'70%', innerHeight:'70%'});
		$("#menu-toggle").click(function(e){
			e.preventDefault();
			$("#wrapper").toggleClass('toggled');
		});
            });

        </script>

 

<div id="wrapper">
                   <!-- Sidebar -->
            <div id="sidebar-wrapper">
             <div id="menubody" style="padding-top:10px">

             <!--div id="tabmenu">
             <ul class="rsat">
                 <li><a class="active" href="index.php">RSAT</a></li>
                 <li><a href="NeAT_home.php" >NeAT</a></li>
                 <!--      <li><a href="http://www.rsat.eu/index_neat.html" target="_top">NeAT</a></li>>
             </ul>
             </div> < /#tabmenu -->
             <div id="content" class="rsat">

                 <div class="menu">
                     <h2>
                         <a href="http://pedagogix-tagc.univ-mrs.fr/rsat/index.php">
                             <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/RSAT_icon.jpg" style="max-width:150px;max-height:60px;" alt="RSAT server" border="0">
                                 </a>
                     </h2>
                 </div>

                 <!--      <p ><h2 style="border-style:solid;border-color:#cc6600;padding:0 10px;background-color:#F6E6CA;"><a href="http://www.eccb14.org/program/tutorials/cis-r" target="_blank">RSAT tutorial<br>at ECCB'14</a></h2></p>>

                  <p ><h2 style="border-style:solid;border-color:#cc6600;padding:0 10px;background-color:#F6E6CA;">
                  <a target='_blank' href="http://rsat.ulb.ac.be/eccb14/" >RSAT tutorial<br>at ECCB'14</a>
                  </h2></p-->
		<div style="padding:60px 0 0 10px;" align="center"><i class="fa fa-bar-chart  fa-lg"></i> <b>
4296</b> <i>organisms</i></div>
                <div>
			<input type="search" id="searchfunc" placeholder="Search" class="searchmenu" onkeypress="searchfunc()" onkeyup="searchfunc()">
		</div>
		<div align="right">
                     <b>New&nbsp;items</b><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" class="new">
                 </div>

                 <div class="menu_expand" onclick="expandAll(&#39;14&#39;)" id="expand"> &gt; view all tools</div>

                 <div class="menu">
                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;1&#39;)" id="heading1">Genomes and genes</div>
                     <div id="menu1" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/supported-organisms.cgi">supported organisms</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/gene-info_form.cgi"> gene information</a>
     

<a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/infer-operons_form.cgi"> infer operons</a><a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/get-orthologs_form.cgi">get orthologs</a>                         
                         
                         
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-genes_form.cgi">random gene selection</a>
                     </div>

                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;2&#39;)" id="heading2">Sequence tools <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu2" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-seq_form.cgi">retrieve sequence</a>
<a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-ensembl-seq_form.cgi">retrieve EnsEMBL seq</a><a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/fetch-sequences_form.php">fetch-sequences from UCSC</a>                         
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-seq-bed_form.cgi">sequences from bed/gff/vcf  <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <!--	  <a class="menu_item" href="http://www.rsat.eu/retrieve-ensembl-seq_form.cgi" >retrieve EnsEMBL seq</a>-->
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/purge-sequence_form.cgi">purge sequence</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-seq_form.cgi">convert sequences</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-seq_form.cgi">random sequences</a>
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;12&#39;)" id="heading12">Matrix tools <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu12" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-matrix_form.cgi">retrieve matrix <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-matrix_form.cgi">convert matrix</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/compare-matrices_form.cgi">compare matrices</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-clustering_form.cgi">matrix-clustering <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-distrib_form.cgi">matrix distrib</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-quality_form.cgi">matrix quality</a>
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;11&#39;)" id="heading11">Build control sets</div>
                     <div id="menu11" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-genes_form.cgi">random gene selection</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-seq_form.cgi">random sequences</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-genome-fragments_form.cgi">random genome fragments</a>
                         <!--	  <a class="menu_item" href="http://www.rsat.eu/random-genome-fragments_form.cgi" >random genome fragments</a>-->
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-motif_form.cgi">random-motif</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/permute-matrix_form.cgi">permute-matrix</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-sites_form.cgi">random-sites</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/implant-sites_form.cgi">implant-sites</a>
                     </div>
                 </div>

                 <div class="menu">
                     <div class="menu_heading_open" onclick="toggleMenu(&#39;3&#39;)" id="heading3">Motif discovery</div>
                     <div id="menu3" class="menu_collapsible" style="display: block;">
                         <a class="menu_separator">strings</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-analysis_form.cgi">oligo-analysis (words)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-diff_form.cgi">oligo-diff (words)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/dyad-analysis_form.cgi">dyad-analysis (spaced pairs)</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/pattern-assembly_form.cgi">pattern assembly</a>
                         <a class="menu_separator">strings with positional biais</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/position-analysis_form.cgi">position-analysis (words)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/local-word-analysis_form.cgi">local-word-analysis (word and spaced pairs)</a>
                         <a class="menu_separator">matrices</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/info-gibbs_form.cgi">info-gibbs</a>
                         <!--<a class="menu_item" href="consensus_form.cgi" >consensus</a>
                         	  <a class="menu_item" href="gibbs_form.cgi" >gibbs</a>-->
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;4&#39;)" id="heading4">Pattern matching <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu4" class="menu_collapsible">
                         <a class="menu_separator">matrices</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-scan_form.cgi">matrix-scan<br>(full options)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-scan-quick_form.cgi">matrix-scan (quick)</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/crer-scan_form.cgi">crer-scan</a>
                         <!--<a class="menu_item" href="matrix-enrichment_form.cgi" >matrix-enrichment <img src="images/onebit_49.png" height="30" class="new"></img></a>-->
                         <!--	  <a class="menu_item" href="patser_form.cgi" >patser [discontinued]</a>-->
                         <!--	  <a class="menu_item" href="genome-scale-patser_form.cgi" >genome-scale patser [discontinued]</a>-->
                         <a class="menu_separator">strings</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/dna-pattern_form.cgi">dna-pattern</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/genome-scale-dna-pattern_form.cgi">genome-scale dna-pattern</a>
                     </div>
                 </div>

                 <div class="menu">
                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;10&#39;)" id="heading10">Comparative genomics&nbsp; <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu10" class="menu_collapsible">
<a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/get-orthologs_form.cgi">get orthologs</a>      
                     
                         <span class="menu_item" style="cursor:default;color:lightgray"> get orthologs-compara </span>                     
                         
                         
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/footprint-discovery_form.cgi">footprint-discovery</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/footprint-scan_form.cgi">footprint-scan <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new">
                         </a>   

                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;13&#39;)" id="heading13">NGS - ChIP-seq</div>
                     <div id="menu13" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/peak-motifs_form.cgi">peak-motifs</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/fetch-sequences_form.php">fetch-sequences from UCSC</a>                        <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-seq-bed_form.cgi">sequences from bed/gff/vcf  <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>

                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/random-genome-fragments_form.cgi">random genome fragments</a>
                         <!--	  <a class="menu_item" href="random-genome-fragments_form.cgi" >random genome fragments</a>-->
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;9&#39;)" id="heading9">Genetic variations (Var-tools) <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></div>
                     <div id="menu9" class="menu_collapsible">
                     
                     <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/variation-info_form.cgi">Variation info <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/retrieve-variation-seq_form.cgi">Retrieve variation sequences <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/variation-scan_form.cgi">Scan variations with motifs <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                                              
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-variations_form.cgi">Convert variation formats <img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/onebit_49.png" height="30" class="new"></a>
                     </div>
                 </div>


                 <div class="menu">
                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;8&#39;)" id="heading8">Conversion/Utilities</div>
                     <div id="menu8" class="menu_collapsible">
                         <a class="menu_separator">Set comparisons / enrichment</a>
                         <!--<a class="menu_item_last" href="compare_classes_form.php?menu=RSAT" >compare classes/clusters</a> -->
                         <a class="menu_separator">Stats</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/classfreq_form.cgi">Frequency distribution</a>
                         <a class="menu_separator">sequences</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-seq_form.cgi">convert sequences</a>
                         <a class="menu_separator">matrices</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-matrix_form.cgi">convert matrix / logo</a>
                         <a class="menu_separator">background models</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/create-background-model_form.cgi">create background</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-background-model_form.cgi">convert background</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/seq-proba_form.cgi">sequence probability</a>
                         <a class="menu_separator">features</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-features_form.cgi">convert features</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/compare-features_form.cgi">compare features</a>
                     </div>


                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;5&#39;)" id="heading5">Drawing</div>
                     <div id="menu5" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/feature-map_form.cgi">feature map</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/feature-map_form2.cgi">feature map 2</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/XYgraph_form.cgi">XY graph</a>
                     </div>


                 </div>

                 <div class="menu">
                     <div class="menu_heading_closed" onclick="toggleMenu(&#39;6&#39;)" id="heading6">SOAP Web services</div>
                     <div id="menu6" class="menu_collapsible">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/htmllink.cgi?title=RSAT%20:%20Web%20services&amp;file=web_services.html">Programmatic interface to RSAT</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/htmllink.cgi?title=RSAT%20:%20Web%20services%20documentation&amp;file=web_services/RSATWS_documentation.xml">WSDL Documentation</a>
                         <!--a class="menu_item" href="web_services/RSATWS.wsdl" target=tools>WSDL</a>
                          <a class="menu_item" href="ws_clients.html" target=tools>Clients</a>
                          <a class="menu_item" href="ws_workflows.html" target=tools>Taverna workflows </a-->
                     </div>

                 </div>

                 <div class="menu">


                     <div class="menu_heading_open" onclick="toggleMenu(&#39;7&#39;)" id="heading7">Help &amp; Contact</div>
                         <div id="menu7" class="menu_collapsible_display">
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/people.php">RSAT team</a>
                         <a class="menu_item" href="http://rsa-tools.github.io/teaching/index.html" target="_blank">Training material</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/tutorials.php">Tutorials</a>

                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/publications.cgi">Publications</a>
                         <a class="menu_item" href="http://pedagogix-tagc.univ-mrs.fr/rsat/credits.php">Credits</a>
                         <a class="menu_item" href="http://teaching.rsat.eu/download-request_form.cgi">Download</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/htmllink.cgi?title=RSAT-motif&amp;file=motif_databases/">Motif databases</a>
                         <a class="menu_item_last" href="http://pedagogix-tagc.univ-mrs.fr/rsat/htmllink.cgi?title=RSAT-Data&amp;file=data/">Data</a>

                        </div>

                         </div>

                        <a target="_top" href="http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-analysis.cgi#" style="font-size:3px;visibility:hidden">bug/dont understand</a>

                    </div>
               </div>

            </div><!-- /#sidebar-wrapper -->
<div class="container" id="page-content-wrapper"><span class="menu-toggle"><a href="http://pedagogix-tagc.univ-mrs.fr/rsat/oligo-analysis.cgi#" id="menu-toggle"><i class="fa fa-bars fa-2x" style="color:green;"></i></a></span><h3 align="center"><a href="http://pedagogix-tagc.univ-mrs.fr/rsat/RSAT_home.cgi">RSAT</a> - oligo-analysis result</h3><blockquote class="information">
<font color="#006600"><b>Information: </b>Uploaded file	iHKG_TypeII_seq_971351913.fasta	152163 bytes</font>
</blockquote><br><hr size="3">
<pre>$RSAT/perl-scripts/convert-seq  -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv -from  fasta -to fasta  -mask non-dna -o $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta</pre>
<b>Server command</b><pre>$RSAT/perl-scripts/purge-sequence -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta -format fasta -o $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged -seqtype dna;$RSAT/perl-scripts/oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged -format fasta  -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible  -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 6 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_6nt.tab; $RSAT/perl-scripts/oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged -format fasta  -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible  -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 7 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_7nt.tab; $RSAT/perl-scripts/oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged -format fasta  -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible  -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 8 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_8nt.tab; cat $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_6nt.tab $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_7nt.tab $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_8nt.tab  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged -format fasta  -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible  -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01
</pre>
<blockquote class="information">
<font color="#006600"><b>Information: </b>One or several button(s) will appear at the bottom of this page, allowing you to send the result as input for a subsequent query.</font>
</blockquote><br><hr size="3">
<h2>Result</h2><pre> oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged -format fasta -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 6 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_6nt.tab
 Citation: van Helden et al. (1998). J Mol Biol 281(5), 827-42. 
 Program version              	1.169
 Quick counting mode          
 Detection of over-represented words (right-tail test)
 Oligomer length              	6
 Input file                   	$RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged
 Input format                 	fasta
 Output file                  	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_6nt.tab
 Discard overlapping matches
 Counted on both strands
 	grouped by pairs of reverse complements
 Background model             	upstream-noorf
 Organism                     	Candida_glabrata
 Background estimation method 	Frequency file
 Expected frequency file      	$RSAT/public_html/data/genomes/Candida_glabrata/oligo-frequencies/6nt_upstream-noorf_Candida_glabrata-noov-2str.freq
 Pseudo-frequency             	0.01
 Pseudo-frequency per oligo   	4.80769230769231e-06
 Sequence type                	DNA
 Nb of sequences              	141
 Sum of sequence lengths      	141000
 discarded residues           	NA (quick mode)	 (other letters than ACGT)
 discarded occurrences        	NA (quick mode)	 (contain discarded residues)
 nb possible positions        	NA (quick mode)
 total oligo occurrences      	139033
 total overlapping occurrences	2240
 total non overlapping occ    	136793
 alphabet size                	4
 nb possible oligomers        	2080
 oligomers tested for significance	2080

 column headers
	1	seq            	oligomer sequence
	2	id             	oligomer identifier
	3	exp_freq       	expected relative frequency
	4	occ            	observed occurrences
	5	exp_occ        	expected occurrences
	6	occ_P          	occurrence probability (binomial)
	7	occ_E          	E-value for occurrences (binomial)
	8	occ_sig        	occurrence significance (binomial)
	9	rank           	rank
	10	ovl_occ        	number of overlapping occurrences (discarded from the count)
	11	forbocc        	forbidden positions (to avoid self-overlap)
</pre>

<table class="sortable" cellspacing="1" cellpadding="3" border="0"><thead><tr valign="top">
<th align="left"><tt>seq</tt></th>
<th align="left"><tt>id</tt></th>
<th align="left"><tt>exp_freq</tt></th>
<th align="left"><tt>occ</tt></th>
<th align="left"><tt>exp_occ</tt></th>
<th align="left"><tt>occ_P</tt></th>
<th align="left"><tt>occ_E</tt></th>
<th align="left"><tt>occ_sig</tt></th>
<th align="left"><tt>rank</tt></th>
<th align="left"><tt>ovl_occ</tt></th>
<th align="left"><tt>forbocc</tt></th>
</tr></thead>
<tbody>

<tr valign="top">
<td><tt>aggggg</tt></td>
<td><tt>aggggg|ccccct</tt></td>
<td><tt>0.0003672889553</tt></td>
<td><tt>106</tt></td>
<td><tt>51.07</tt></td>
<td><tt>1.3e-11</tt></td>
<td><tt>2.6e-08</tt></td>
<td><tt>7.58</tt></td>
<td><tt>1</tt></td>
<td><tt>0</tt></td>
<td><tt>530</tt></td>
</tr>

<tr valign="top">
<td><tt>gagtca</tt></td>
<td><tt>gagtca|tgactc</tt></td>
<td><tt>0.0003360840466</tt></td>
<td><tt>93</tt></td>
<td><tt>46.73</tt></td>
<td><tt>1.6e-09</tt></td>
<td><tt>3.3e-06</tt></td>
<td><tt>5.48</tt></td>
<td><tt>2</tt></td>
<td><tt>0</tt></td>
<td><tt>465</tt></td>
</tr>

<tr valign="top">
<td><tt>ggggta</tt></td>
<td><tt>ggggta|tacccc</tt></td>
<td><tt>0.0001803747040</tt></td>
<td><tt>57</tt></td>
<td><tt>25.08</tt></td>
<td><tt>3.2e-08</tt></td>
<td><tt>6.6e-05</tt></td>
<td><tt>4.18</tt></td>
<td><tt>3</tt></td>
<td><tt>0</tt></td>
<td><tt>285</tt></td>
</tr>

<tr valign="top">
<td><tt>aagggg</tt></td>
<td><tt>aagggg|cccctt</tt></td>
<td><tt>0.0003975482608</tt></td>
<td><tt>100</tt></td>
<td><tt>55.27</tt></td>
<td><tt>4.1e-08</tt></td>
<td><tt>8.5e-05</tt></td>
<td><tt>4.07</tt></td>
<td><tt>4</tt></td>
<td><tt>0</tt></td>
<td><tt>500</tt></td>
</tr>

<tr valign="top">
<td><tt>gaagaa</tt></td>
<td><tt>gaagaa|ttcttc</tt></td>
<td><tt>0.0009715294608</tt></td>
<td><tt>199</tt></td>
<td><tt>135.07</tt></td>
<td><tt>1.6e-07</tt></td>
<td><tt>3.3e-04</tt></td>
<td><tt>3.48</tt></td>
<td><tt>5</tt></td>
<td><tt>12</tt></td>
<td><tt>995</tt></td>
</tr>

<tr valign="top">
<td><tt>caccag</tt></td>
<td><tt>caccag|ctggtg</tt></td>
<td><tt>0.0003357688456</tt></td>
<td><tt>86</tt></td>
<td><tt>46.68</tt></td>
<td><tt>1.6e-07</tt></td>
<td><tt>3.4e-04</tt></td>
<td><tt>3.47</tt></td>
<td><tt>6</tt></td>
<td><tt>0</tt></td>
<td><tt>430</tt></td>
</tr>

<tr valign="top">
<td><tt>catccc</tt></td>
<td><tt>catccc|gggatg</tt></td>
<td><tt>0.0003244216059</tt></td>
<td><tt>81</tt></td>
<td><tt>45.11</tt></td>
<td><tt>9.5e-07</tt></td>
<td><tt>2.0e-03</tt></td>
<td><tt>2.70</tt></td>
<td><tt>7</tt></td>
<td><tt>0</tt></td>
<td><tt>405</tt></td>
</tr>

<tr valign="top">
<td><tt>gcacca</tt></td>
<td><tt>gcacca|tggtgc</tt></td>
<td><tt>0.0003222151983</tt></td>
<td><tt>80</tt></td>
<td><tt>44.80</tt></td>
<td><tt>1.4e-06</tt></td>
<td><tt>2.8e-03</tt></td>
<td><tt>2.55</tt></td>
<td><tt>8</tt></td>
<td><tt>0</tt></td>
<td><tt>400</tt></td>
</tr>

<tr valign="top">
<td><tt>accagc</tt></td>
<td><tt>accagc|gctggt</tt></td>
<td><tt>0.0003326168346</tt></td>
<td><tt>81</tt></td>
<td><tt>46.24</tt></td>
<td><tt>2.4e-06</tt></td>
<td><tt>4.9e-03</tt></td>
<td><tt>2.31</tt></td>
<td><tt>9</tt></td>
<td><tt>0</tt></td>
<td><tt>405</tt></td>
</tr>

<tr valign="top">
<td><tt>cccttc</tt></td>
<td><tt>cccttc|gaaggg</tt></td>
<td><tt>0.0005217374936</tt></td>
<td><tt>115</tt></td>
<td><tt>72.54</tt></td>
<td><tt>2.6e-06</tt></td>
<td><tt>5.4e-03</tt></td>
<td><tt>2.27</tt></td>
<td><tt>10</tt></td>
<td><tt>2</tt></td>
<td><tt>575</tt></td>
</tr>

<tr valign="top">
<td><tt>aagatg</tt></td>
<td><tt>aagatg|catctt</tt></td>
<td><tt>0.0005166942760</tt></td>
<td><tt>114</tt></td>
<td><tt>71.84</tt></td>
<td><tt>2.7e-06</tt></td>
<td><tt>5.6e-03</tt></td>
<td><tt>2.25</tt></td>
<td><tt>11</tt></td>
<td><tt>0</tt></td>
<td><tt>570</tt></td>
</tr>

<tr valign="top">
<td><tt>gactcc</tt></td>
<td><tt>gactcc|ggagtc</tt></td>
<td><tt>0.0001687122634</tt></td>
<td><tt>49</tt></td>
<td><tt>23.46</tt></td>
<td><tt>2.8e-06</tt></td>
<td><tt>5.7e-03</tt></td>
<td><tt>2.24</tt></td>
<td><tt>12</tt></td>
<td><tt>0</tt></td>
<td><tt>245</tt></td>
</tr>

<tr valign="top">
<td><tt>agaaga</tt></td>
<td><tt>agaaga|tcttct</tt></td>
<td><tt>0.0009661710421</tt></td>
<td><tt>188</tt></td>
<td><tt>134.33</tt></td>
<td><tt>7.1e-06</tt></td>
<td><tt>1.5e-02</tt></td>
<td><tt>1.83</tt></td>
<td><tt>13</tt></td>
<td><tt>11</tt></td>
<td><tt>940</tt></td>
</tr>

<tr valign="top">
<td><tt>cccccc</tt></td>
<td><tt>cccccc|gggggg</tt></td>
<td><tt>0.0005630288374</tt></td>
<td><tt>120</tt></td>
<td><tt>78.28</tt></td>
<td><tt>7.2e-06</tt></td>
<td><tt>1.5e-02</tt></td>
<td><tt>1.83</tt></td>
<td><tt>14</tt></td>
<td><tt>152</tt></td>
<td><tt>600</tt></td>
</tr>

<tr valign="top">
<td><tt>gtggaa</tt></td>
<td><tt>gtggaa|ttccac</tt></td>
<td><tt>0.0004946301990</tt></td>
<td><tt>108</tt></td>
<td><tt>68.77</tt></td>
<td><tt>7.4e-06</tt></td>
<td><tt>1.5e-02</tt></td>
<td><tt>1.81</tt></td>
<td><tt>15</tt></td>
<td><tt>0</tt></td>
<td><tt>540</tt></td>
</tr>

<tr valign="top">
<td><tt>ccacca</tt></td>
<td><tt>ccacca|tggtgg</tt></td>
<td><tt>0.0003953418531</tt></td>
<td><tt>90</tt></td>
<td><tt>54.97</tt></td>
<td><tt>9e-06</tt></td>
<td><tt>1.9e-02</tt></td>
<td><tt>1.73</tt></td>
<td><tt>16</tt></td>
<td><tt>7</tt></td>
<td><tt>450</tt></td>
</tr>

<tr valign="top">
<td><tt>gaaccc</tt></td>
<td><tt>gaaccc|gggttc</tt></td>
<td><tt>0.0002544469621</tt></td>
<td><tt>64</tt></td>
<td><tt>35.38</tt></td>
<td><tt>9.6e-06</tt></td>
<td><tt>2.0e-02</tt></td>
<td><tt>1.70</tt></td>
<td><tt>17</tt></td>
<td><tt>0</tt></td>
<td><tt>320</tt></td>
</tr>

<tr valign="top">
<td><tt>caccac</tt></td>
<td><tt>caccac|gtggtg</tt></td>
<td><tt>0.0003594089279</tt></td>
<td><tt>83</tt></td>
<td><tt>49.97</tt></td>
<td><tt>1.2e-05</tt></td>
<td><tt>2.5e-02</tt></td>
<td><tt>1.61</tt></td>
<td><tt>18</tt></td>
<td><tt>6</tt></td>
<td><tt>415</tt></td>
</tr>

<tr valign="top">
<td><tt>accacc</tt></td>
<td><tt>accacc|ggtggt</tt></td>
<td><tt>0.0002995207192</tt></td>
<td><tt>72</tt></td>
<td><tt>41.64</tt></td>
<td><tt>1.2e-05</tt></td>
<td><tt>2.6e-02</tt></td>
<td><tt>1.59</tt></td>
<td><tt>19</tt></td>
<td><tt>4</tt></td>
<td><tt>360</tt></td>
</tr>

<tr valign="top">
<td><tt>ggaaga</tt></td>
<td><tt>ggaaga|tcttcc</tt></td>
<td><tt>0.0005438015704</tt></td>
<td><tt>115</tt></td>
<td><tt>75.61</tt></td>
<td><tt>1.5e-05</tt></td>
<td><tt>3.1e-02</tt></td>
<td><tt>1.51</tt></td>
<td><tt>20</tt></td>
<td><tt>0</tt></td>
<td><tt>575</tt></td>
</tr>

<tr valign="top">
<td><tt>ccagcc</tt></td>
<td><tt>ccagcc|ggctgg</tt></td>
<td><tt>0.0001961347590</tt></td>
<td><tt>52</tt></td>
<td><tt>27.27</tt></td>
<td><tt>1.6e-05</tt></td>
<td><tt>3.4e-02</tt></td>
<td><tt>1.47</tt></td>
<td><tt>21</tt></td>
<td><tt>0</tt></td>
<td><tt>260</tt></td>
</tr>

<tr valign="top">
<td><tt>ccagcg</tt></td>
<td><tt>ccagcg|cgctgg</tt></td>
<td><tt>0.0001561042195</tt></td>
<td><tt>44</tt></td>
<td><tt>21.70</tt></td>
<td><tt>1.7e-05</tt></td>
<td><tt>3.6e-02</tt></td>
<td><tt>1.45</tt></td>
<td><tt>22</tt></td>
<td><tt>0</tt></td>
<td><tt>220</tt></td>
</tr>

<tr valign="top">
<td><tt>caccgg</tt></td>
<td><tt>caccgg|ccggtg</tt></td>
<td><tt>0.0001277361206</tt></td>
<td><tt>38</tt></td>
<td><tt>17.76</tt></td>
<td><tt>2e-05</tt></td>
<td><tt>4.2e-02</tt></td>
<td><tt>1.38</tt></td>
<td><tt>23</tt></td>
<td><tt>3</tt></td>
<td><tt>190</tt></td>
</tr>

<tr valign="top">
<td><tt>ggggga</tt></td>
<td><tt>ggggga|tccccc</tt></td>
<td><tt>0.0005428559671</tt></td>
<td><tt>114</tt></td>
<td><tt>75.47</tt></td>
<td><tt>2.2e-05</tt></td>
<td><tt>4.5e-02</tt></td>
<td><tt>1.35</tt></td>
<td><tt>24</tt></td>
<td><tt>0</tt></td>
<td><tt>570</tt></td>
</tr>

<tr valign="top">
<td><tt>acaccc</tt></td>
<td><tt>acaccc|gggtgt</tt></td>
<td><tt>0.0003291496225</tt></td>
<td><tt>76</tt></td>
<td><tt>45.76</tt></td>
<td><tt>2.7e-05</tt></td>
<td><tt>5.6e-02</tt></td>
<td><tt>1.25</tt></td>
<td><tt>25</tt></td>
<td><tt>0</tt></td>
<td><tt>380</tt></td>
</tr>

<tr valign="top">
<td><tt>gaggga</tt></td>
<td><tt>gaggga|tccctc</tt></td>
<td><tt>0.0004252859574</tt></td>
<td><tt>93</tt></td>
<td><tt>59.13</tt></td>
<td><tt>2.8e-05</tt></td>
<td><tt>5.9e-02</tt></td>
<td><tt>1.23</tt></td>
<td><tt>26</tt></td>
<td><tt>2</tt></td>
<td><tt>465</tt></td>
</tr>

<tr valign="top">
<td><tt>ccaccc</tt></td>
<td><tt>ccaccc|gggtgg</tt></td>
<td><tt>0.0002598053809</tt></td>
<td><tt>63</tt></td>
<td><tt>36.12</tt></td>
<td><tt>3.2e-05</tt></td>
<td><tt>6.6e-02</tt></td>
<td><tt>1.18</tt></td>
<td><tt>27</tt></td>
<td><tt>0</tt></td>
<td><tt>315</tt></td>
</tr>

<tr valign="top">
<td><tt>acgagg</tt></td>
<td><tt>acgagg|cctcgt</tt></td>
<td><tt>0.0001469633876</tt></td>
<td><tt>41</tt></td>
<td><tt>20.43</tt></td>
<td><tt>4e-05</tt></td>
<td><tt>8.4e-02</tt></td>
<td><tt>1.08</tt></td>
<td><tt>28</tt></td>
<td><tt>0</tt></td>
<td><tt>205</tt></td>
</tr>

<tr valign="top">
<td><tt>gacgca</tt></td>
<td><tt>gacgca|tgcgtc</tt></td>
<td><tt>0.0001472785887</tt></td>
<td><tt>41</tt></td>
<td><tt>20.48</tt></td>
<td><tt>4.2e-05</tt></td>
<td><tt>8.8e-02</tt></td>
<td><tt>1.06</tt></td>
<td><tt>29</tt></td>
<td><tt>0</tt></td>
<td><tt>205</tt></td>
</tr>

<tr valign="top">
<td><tt>cccgag</tt></td>
<td><tt>cccgag|ctcggg</tt></td>
<td><tt>0.0001037808372</tt></td>
<td><tt>32</tt></td>
<td><tt>14.43</tt></td>
<td><tt>4.5e-05</tt></td>
<td><tt>9.3e-02</tt></td>
<td><tt>1.03</tt></td>
<td><tt>30</tt></td>
<td><tt>0</tt></td>
<td><tt>160</tt></td>
</tr>

<tr valign="top">
<td><tt>ccagta</tt></td>
<td><tt>ccagta|tactgg</tt></td>
<td><tt>0.0004372635992</tt></td>
<td><tt>94</tt></td>
<td><tt>60.79</tt></td>
<td><tt>4.7e-05</tt></td>
<td><tt>9.8e-02</tt></td>
<td><tt>1.01</tt></td>
<td><tt>31</tt></td>
<td><tt>0</tt></td>
<td><tt>470</tt></td>
</tr>

<tr valign="top">
<td><tt>ggtgga</tt></td>
<td><tt>ggtgga|tccacc</tt></td>
<td><tt>0.0002689462126</tt></td>
<td><tt>64</tt></td>
<td><tt>37.39</tt></td>
<td><tt>4.7e-05</tt></td>
<td><tt>9.9e-02</tt></td>
<td><tt>1.01</tt></td>
<td><tt>32</tt></td>
<td><tt>0</tt></td>
<td><tt>320</tt></td>
</tr>

<tr valign="top">
<td><tt>cgcacc</tt></td>
<td><tt>cgcacc|ggtgcg</tt></td>
<td><tt>0.0001242689086</tt></td>
<td><tt>36</tt></td>
<td><tt>17.28</tt></td>
<td><tt>5.5e-05</tt></td>
<td><tt>1.1e-01</tt></td>
<td><tt>0.94</tt></td>
<td><tt>33</tt></td>
<td><tt>1</tt></td>
<td><tt>180</tt></td>
</tr>

<tr valign="top">
<td><tt>accccg</tt></td>
<td><tt>accccg|cggggt</tt></td>
<td><tt>0.0001050416416</tt></td>
<td><tt>32</tt></td>
<td><tt>14.60</tt></td>
<td><tt>5.6e-05</tt></td>
<td><tt>1.2e-01</tt></td>
<td><tt>0.94</tt></td>
<td><tt>34</tt></td>
<td><tt>0</tt></td>
<td><tt>160</tt></td>
</tr>

<tr valign="top">
<td><tt>caccca</tt></td>
<td><tt>caccca|tgggtg</tt></td>
<td><tt>0.0003376600522</tt></td>
<td><tt>76</tt></td>
<td><tt>46.95</tt></td>
<td><tt>5.9e-05</tt></td>
<td><tt>1.2e-01</tt></td>
<td><tt>0.91</tt></td>
<td><tt>35</tt></td>
<td><tt>0</tt></td>
<td><tt>380</tt></td>
</tr>

<tr valign="top">
<td><tt>ctcagc</tt></td>
<td><tt>ctcagc|gctgag</tt></td>
<td><tt>0.0002443605270</tt></td>
<td><tt>59</tt></td>
<td><tt>33.97</tt></td>
<td><tt>6.2e-05</tt></td>
<td><tt>1.3e-01</tt></td>
<td><tt>0.89</tt></td>
<td><tt>36</tt></td>
<td><tt>1</tt></td>
<td><tt>295</tt></td>
</tr>

<tr valign="top">
<td><tt>cccgca</tt></td>
<td><tt>cccgca|tgcggg</tt></td>
<td><tt>0.0002349044941</tt></td>
<td><tt>57</tt></td>
<td><tt>32.66</tt></td>
<td><tt>7.2e-05</tt></td>
<td><tt>1.5e-01</tt></td>
<td><tt>0.83</tt></td>
<td><tt>37</tt></td>
<td><tt>0</tt></td>
<td><tt>285</tt></td>
</tr>

<tr valign="top">
<td><tt>cttgtc</tt></td>
<td><tt>cttgtc|gacaag</tt></td>
<td><tt>0.0004445132245</tt></td>
<td><tt>94</tt></td>
<td><tt>61.80</tt></td>
<td><tt>8.3e-05</tt></td>
<td><tt>1.7e-01</tt></td>
<td><tt>0.76</tt></td>
<td><tt>38</tt></td>
<td><tt>0</tt></td>
<td><tt>470</tt></td>
</tr>

<tr valign="top">
<td><tt>cttccc</tt></td>
<td><tt>cttccc|gggaag</tt></td>
<td><tt>0.0005620832341</tt></td>
<td><tt>114</tt></td>
<td><tt>78.15</tt></td>
<td><tt>8.4e-05</tt></td>
<td><tt>1.8e-01</tt></td>
<td><tt>0.76</tt></td>
<td><tt>39</tt></td>
<td><tt>4</tt></td>
<td><tt>570</tt></td>
</tr>

<tr valign="top">
<td><tt>acccct</tt></td>
<td><tt>acccct|aggggt</tt></td>
<td><tt>0.0001794291008</tt></td>
<td><tt>46</tt></td>
<td><tt>24.95</tt></td>
<td><tt>0.00010</tt></td>
<td><tt>2.1e-01</tt></td>
<td><tt>0.68</tt></td>
<td><tt>40</tt></td>
<td><tt>0</tt></td>
<td><tt>230</tt></td>
</tr>

<tr valign="top">
<td><tt>cttcca</tt></td>
<td><tt>cttcca|tggaag</tt></td>
<td><tt>0.0006276450625</tt></td>
<td><tt>124</tt></td>
<td><tt>87.26</tt></td>
<td><tt>0.00012</tt></td>
<td><tt>2.6e-01</tt></td>
<td><tt>0.59</tt></td>
<td><tt>41</tt></td>
<td><tt>0</tt></td>
<td><tt>620</tt></td>
</tr>

<tr valign="top">
<td><tt>cctccc</tt></td>
<td><tt>cctccc|gggagg</tt></td>
<td><tt>0.0003868314234</tt></td>
<td><tt>83</tt></td>
<td><tt>53.78</tt></td>
<td><tt>0.00013</tt></td>
<td><tt>2.7e-01</tt></td>
<td><tt>0.56</tt></td>
<td><tt>42</tt></td>
<td><tt>3</tt></td>
<td><tt>415</tt></td>
</tr>

<tr valign="top">
<td><tt>ccctta</tt></td>
<td><tt>ccctta|taaggg</tt></td>
<td><tt>0.0003244216059</tt></td>
<td><tt>72</tt></td>
<td><tt>45.11</tt></td>
<td><tt>0.00014</tt></td>
<td><tt>2.8e-01</tt></td>
<td><tt>0.55</tt></td>
<td><tt>43</tt></td>
<td><tt>0</tt></td>
<td><tt>360</tt></td>
</tr>

<tr valign="top">
<td><tt>cagggg</tt></td>
<td><tt>cagggg|cccctg</tt></td>
<td><tt>0.0001406593657</tt></td>
<td><tt>38</tt></td>
<td><tt>19.56</tt></td>
<td><tt>0.00014</tt></td>
<td><tt>2.9e-01</tt></td>
<td><tt>0.53</tt></td>
<td><tt>44</tt></td>
<td><tt>0</tt></td>
<td><tt>190</tt></td>
</tr>

<tr valign="top">
<td><tt>aagaag</tt></td>
<td><tt>aagaag|cttctt</tt></td>
<td><tt>0.0010011583640</tt></td>
<td><tt>184</tt></td>
<td><tt>139.19</tt></td>
<td><tt>0.00016</tt></td>
<td><tt>3.4e-01</tt></td>
<td><tt>0.47</tt></td>
<td><tt>45</tt></td>
<td><tt>10</tt></td>
<td><tt>920</tt></td>
</tr>

<tr valign="top">
<td><tt>cgagga</tt></td>
<td><tt>cgagga|tcctcg</tt></td>
<td><tt>0.0002437301248</tt></td>
<td><tt>57</tt></td>
<td><tt>33.89</tt></td>
<td><tt>0.00018</tt></td>
<td><tt>3.7e-01</tt></td>
<td><tt>0.43</tt></td>
<td><tt>46</tt></td>
<td><tt>1</tt></td>
<td><tt>285</tt></td>
</tr>

<tr valign="top">
<td><tt>accccc</tt></td>
<td><tt>accccc|gggggt</tt></td>
<td><tt>0.0003124439642</tt></td>
<td><tt>69</tt></td>
<td><tt>43.44</tt></td>
<td><tt>0.00021</tt></td>
<td><tt>4.4e-01</tt></td>
<td><tt>0.36</tt></td>
<td><tt>47</tt></td>
<td><tt>0</tt></td>
<td><tt>345</tt></td>
</tr>

<tr valign="top">
<td><tt>cgcagc</tt></td>
<td><tt>cgcagc|gctgcg</tt></td>
<td><tt>0.0001696578667</tt></td>
<td><tt>43</tt></td>
<td><tt>23.59</tt></td>
<td><tt>0.00021</tt></td>
<td><tt>4.4e-01</tt></td>
<td><tt>0.36</tt></td>
<td><tt>48</tt></td>
<td><tt>0</tt></td>
<td><tt>215</tt></td>
</tr>

<tr valign="top">
<td><tt>tctcca</tt></td>
<td><tt>tctcca|tggaga</tt></td>
<td><tt>0.0003814730048</tt></td>
<td><tt>81</tt></td>
<td><tt>53.04</tt></td>
<td><tt>0.00021</tt></td>
<td><tt>4.4e-01</tt></td>
<td><tt>0.35</tt></td>
<td><tt>49</tt></td>
<td><tt>0</tt></td>
<td><tt>405</tt></td>
</tr>

<tr valign="top">
<td><tt>ccctcc</tt></td>
<td><tt>ccctcc|ggaggg</tt></td>
<td><tt>0.0003887226300</tt></td>
<td><tt>82</tt></td>
<td><tt>54.05</tt></td>
<td><tt>0.00024</tt></td>
<td><tt>5.0e-01</tt></td>
<td><tt>0.30</tt></td>
<td><tt>50</tt></td>
<td><tt>5</tt></td>
<td><tt>410</tt></td>
</tr>

</tbody><tfoot></tfoot></table><pre> Host name	pedagogix
 Job started	2020-01-31.101329
 Job done	2020-01-31.101331
 Seconds	1.2
	user	1.21
	system	0.06
	cuser	0.47
	csystem	0.04
 oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged -format fasta -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 7 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_7nt.tab
 Citation: van Helden et al. (1998). J Mol Biol 281(5), 827-42. 
 Program version              	1.169
 Quick counting mode          
 Detection of over-represented words (right-tail test)
 Oligomer length              	7
 Input file                   	$RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged
 Input format                 	fasta
 Output file                  	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_7nt.tab
 Discard overlapping matches
 Counted on both strands
 	grouped by pairs of reverse complements
 Background model             	upstream-noorf
 Organism                     	Candida_glabrata
 Background estimation method 	Frequency file
 Expected frequency file      	$RSAT/public_html/data/genomes/Candida_glabrata/oligo-frequencies/7nt_upstream-noorf_Candida_glabrata-noov-2str.freq
 Pseudo-frequency             	0.01
 Pseudo-frequency per oligo   	1.220703125e-06
 Sequence type                	DNA
 Nb of sequences              	141
 Sum of sequence lengths      	141000
 discarded residues           	NA (quick mode)	 (other letters than ACGT)
 discarded occurrences        	NA (quick mode)	 (contain discarded residues)
 nb possible positions        	NA (quick mode)
 total oligo occurrences      	138821
 total overlapping occurrences	1825
 total non overlapping occ    	136996
 alphabet size                	4
 nb possible oligomers        	8192
 oligomers tested for significance	8192

 column headers
	1	seq            	oligomer sequence
	2	id             	oligomer identifier
	3	exp_freq       	expected relative frequency
	4	occ            	observed occurrences
	5	exp_occ        	expected occurrences
	6	occ_P          	occurrence probability (binomial)
	7	occ_E          	E-value for occurrences (binomial)
	8	occ_sig        	occurrence significance (binomial)
	9	rank           	rank
	10	ovl_occ        	number of overlapping occurrences (discarded from the count)
	11	forbocc        	forbidden positions (to avoid self-overlap)
</pre>

<table class="sortable" cellspacing="1" cellpadding="3" border="0"><thead><tr valign="top">
<th align="left"><tt>seq</tt></th>
<th align="left"><tt>id</tt></th>
<th align="left"><tt>exp_freq</tt></th>
<th align="left"><tt>occ</tt></th>
<th align="left"><tt>exp_occ</tt></th>
<th align="left"><tt>occ_P</tt></th>
<th align="left"><tt>occ_E</tt></th>
<th align="left"><tt>occ_sig</tt></th>
<th align="left"><tt>rank</tt></th>
<th align="left"><tt>ovl_occ</tt></th>
<th align="left"><tt>forbocc</tt></th>
</tr></thead>
<tbody>

<tr valign="top">
<td><tt>caccagc</tt></td>
<td><tt>caccagc|gctggtg</tt></td>
<td><tt>0.0000833772781</tt></td>
<td><tt>37</tt></td>
<td><tt>11.57</tt></td>
<td><tt>2.2e-09</tt></td>
<td><tt>1.8e-05</tt></td>
<td><tt>4.75</tt></td>
<td><tt>1</tt></td>
<td><tt>0</tt></td>
<td><tt>222</tt></td>
</tr>

<tr valign="top">
<td><tt>atgactc</tt></td>
<td><tt>atgactc|gagtcat</tt></td>
<td><tt>0.0000972806984</tt></td>
<td><tt>40</tt></td>
<td><tt>13.50</tt></td>
<td><tt>4.1e-09</tt></td>
<td><tt>3.4e-05</tt></td>
<td><tt>4.47</tt></td>
<td><tt>2</tt></td>
<td><tt>0</tt></td>
<td><tt>240</tt></td>
</tr>

<tr valign="top">
<td><tt>cttcttc</tt></td>
<td><tt>cttcttc|gaagaag</tt></td>
<td><tt>0.0002126158903</tt></td>
<td><tt>64</tt></td>
<td><tt>29.52</tt></td>
<td><tt>2.6e-08</tt></td>
<td><tt>2.1e-04</tt></td>
<td><tt>3.67</tt></td>
<td><tt>3</tt></td>
<td><tt>7</tt></td>
<td><tt>384</tt></td>
</tr>

<tr valign="top">
<td><tt>agggggg</tt></td>
<td><tt>agggggg|cccccct</tt></td>
<td><tt>0.0002122999035</tt></td>
<td><tt>63</tt></td>
<td><tt>29.47</tt></td>
<td><tt>5.5e-08</tt></td>
<td><tt>4.5e-04</tt></td>
<td><tt>3.35</tt></td>
<td><tt>4</tt></td>
<td><tt>0</tt></td>
<td><tt>378</tt></td>
</tr>

<tr valign="top">
<td><tt>ccccccc</tt></td>
<td><tt>ccccccc|ggggggg</tt></td>
<td><tt>0.0003001442413</tt></td>
<td><tt>76</tt></td>
<td><tt>41.67</tt></td>
<td><tt>1.2e-06</tt></td>
<td><tt>9.5e-03</tt></td>
<td><tt>2.02</tt></td>
<td><tt>5</tt></td>
<td><tt>79</tt></td>
<td><tt>456</tt></td>
</tr>

<tr valign="top">
<td><tt>acaccca</tt></td>
<td><tt>acaccca|tgggtgt</tt></td>
<td><tt>0.0001307753021</tt></td>
<td><tt>41</tt></td>
<td><tt>18.15</tt></td>
<td><tt>2.8e-06</tt></td>
<td><tt>2.3e-02</tt></td>
<td><tt>1.64</tt></td>
<td><tt>6</tt></td>
<td><tt>0</tt></td>
<td><tt>246</tt></td>
</tr>

<tr valign="top">
<td><tt>aaggggg</tt></td>
<td><tt>aaggggg|ccccctt</tt></td>
<td><tt>0.0001361470781</tt></td>
<td><tt>41</tt></td>
<td><tt>18.90</tt></td>
<td><tt>7.2e-06</tt></td>
<td><tt>5.9e-02</tt></td>
<td><tt>1.23</tt></td>
<td><tt>7</tt></td>
<td><tt>0</tt></td>
<td><tt>246</tt></td>
</tr>

<tr valign="top">
<td><tt>ccccttc</tt></td>
<td><tt>ccccttc|gaagggg</tt></td>
<td><tt>0.0001721695764</tt></td>
<td><tt>48</tt></td>
<td><tt>23.90</tt></td>
<td><tt>9.4e-06</tt></td>
<td><tt>7.7e-02</tt></td>
<td><tt>1.11</tt></td>
<td><tt>8</tt></td>
<td><tt>1</tt></td>
<td><tt>288</tt></td>
</tr>

<tr valign="top">
<td><tt>ctggtgc</tt></td>
<td><tt>ctggtgc|gcaccag</tt></td>
<td><tt>0.0000653660289</tt></td>
<td><tt>25</tt></td>
<td><tt>9.07</tt></td>
<td><tt>9.9e-06</tt></td>
<td><tt>8.1e-02</tt></td>
<td><tt>1.09</tt></td>
<td><tt>9</tt></td>
<td><tt>0</tt></td>
<td><tt>150</tt></td>
</tr>

<tr valign="top">
<td><tt>caccacc</tt></td>
<td><tt>caccacc|ggtggtg</tt></td>
<td><tt>0.0000795854362</tt></td>
<td><tt>28</tt></td>
<td><tt>11.05</tt></td>
<td><tt>1.4e-05</tt></td>
<td><tt>1.1e-01</tt></td>
<td><tt>0.95</tt></td>
<td><tt>10</tt></td>
<td><tt>0</tt></td>
<td><tt>168</tt></td>
</tr>

<tr valign="top">
<td><tt>gagggaa</tt></td>
<td><tt>gagggaa|ttccctc</tt></td>
<td><tt>0.0001557382615</tt></td>
<td><tt>44</tt></td>
<td><tt>21.62</tt></td>
<td><tt>1.6e-05</tt></td>
<td><tt>1.3e-01</tt></td>
<td><tt>0.89</tt></td>
<td><tt>11</tt></td>
<td><tt>0</tt></td>
<td><tt>264</tt></td>
</tr>

<tr valign="top">
<td><tt>cgcacca</tt></td>
<td><tt>cgcacca|tggtgcg</tt></td>
<td><tt>0.0000508306349</tt></td>
<td><tt>21</tt></td>
<td><tt>7.06</tt></td>
<td><tt>1.6e-05</tt></td>
<td><tt>1.3e-01</tt></td>
<td><tt>0.88</tt></td>
<td><tt>12</tt></td>
<td><tt>0</tt></td>
<td><tt>126</tt></td>
</tr>

<tr valign="top">
<td><tt>cccccta</tt></td>
<td><tt>cccccta|taggggg</tt></td>
<td><tt>0.0000400870828</tt></td>
<td><tt>18</tt></td>
<td><tt>5.56</tt></td>
<td><tt>2.2e-05</tt></td>
<td><tt>1.8e-01</tt></td>
<td><tt>0.74</tt></td>
<td><tt>13</tt></td>
<td><tt>0</tt></td>
<td><tt>108</tt></td>
</tr>

<tr valign="top">
<td><tt>gggatga</tt></td>
<td><tt>gggatga|tcatccc</tt></td>
<td><tt>0.0000821133308</tt></td>
<td><tt>28</tt></td>
<td><tt>11.40</tt></td>
<td><tt>2.3e-05</tt></td>
<td><tt>1.9e-01</tt></td>
<td><tt>0.72</tt></td>
<td><tt>14</tt></td>
<td><tt>0</tt></td>
<td><tt>168</tt></td>
</tr>

<tr valign="top">
<td><tt>ccaccac</tt></td>
<td><tt>ccaccac|gtggtgg</tt></td>
<td><tt>0.0000915929356</tt></td>
<td><tt>30</tt></td>
<td><tt>12.72</tt></td>
<td><tt>2.6e-05</tt></td>
<td><tt>2.1e-01</tt></td>
<td><tt>0.68</tt></td>
<td><tt>15</tt></td>
<td><tt>3</tt></td>
<td><tt>180</tt></td>
</tr>

<tr valign="top">
<td><tt>ggagtca</tt></td>
<td><tt>ggagtca|tgactcc</tt></td>
<td><tt>0.0000609422134</tt></td>
<td><tt>23</tt></td>
<td><tt>8.46</tt></td>
<td><tt>2.7e-05</tt></td>
<td><tt>2.2e-01</tt></td>
<td><tt>0.66</tt></td>
<td><tt>16</tt></td>
<td><tt>0</tt></td>
<td><tt>138</tt></td>
</tr>

<tr valign="top">
<td><tt>catcttc</tt></td>
<td><tt>catcttc|gaagatg</tt></td>
<td><tt>0.0001115001057</tt></td>
<td><tt>34</tt></td>
<td><tt>15.48</tt></td>
<td><tt>3.2e-05</tt></td>
<td><tt>2.6e-01</tt></td>
<td><tt>0.58</tt></td>
<td><tt>17</tt></td>
<td><tt>0</tt></td>
<td><tt>204</tt></td>
</tr>

<tr valign="top">
<td><tt>acgagga</tt></td>
<td><tt>acgagga|tcctcgt</tt></td>
<td><tt>0.0000492507008</tt></td>
<td><tt>20</tt></td>
<td><tt>6.84</tt></td>
<td><tt>3.2e-05</tt></td>
<td><tt>2.6e-01</tt></td>
<td><tt>0.58</tt></td>
<td><tt>18</tt></td>
<td><tt>0</tt></td>
<td><tt>120</tt></td>
</tr>

<tr valign="top">
<td><tt>ctgtaag</tt></td>
<td><tt>ctgtaag|cttacag</tt></td>
<td><tt>0.0001203477369</tt></td>
<td><tt>35</tt></td>
<td><tt>16.71</tt></td>
<td><tt>6.2e-05</tt></td>
<td><tt>5.1e-01</tt></td>
<td><tt>0.29</tt></td>
<td><tt>19</tt></td>
<td><tt>0</tt></td>
<td><tt>210</tt></td>
</tr>

<tr valign="top">
<td><tt>gggggta</tt></td>
<td><tt>gggggta|taccccc</tt></td>
<td><tt>0.0000568343847</tt></td>
<td><tt>21</tt></td>
<td><tt>7.89</tt></td>
<td><tt>7.8e-05</tt></td>
<td><tt>6.4e-01</tt></td>
<td><tt>0.20</tt></td>
<td><tt>20</tt></td>
<td><tt>0</tt></td>
<td><tt>126</tt></td>
</tr>

<tr valign="top">
<td><tt>cgacacc</tt></td>
<td><tt>cgacacc|ggtgtcg</tt></td>
<td><tt>0.0000255516888</tt></td>
<td><tt>13</tt></td>
<td><tt>3.55</tt></td>
<td><tt>8.7e-05</tt></td>
<td><tt>7.1e-01</tt></td>
<td><tt>0.15</tt></td>
<td><tt>21</tt></td>
<td><tt>0</tt></td>
<td><tt>78</tt></td>
</tr>

<tr valign="top">
<td><tt>agggatg</tt></td>
<td><tt>agggatg|catccct</tt></td>
<td><tt>0.0000843252386</tt></td>
<td><tt>27</tt></td>
<td><tt>11.71</tt></td>
<td><tt>9e-05</tt></td>
<td><tt>7.4e-01</tt></td>
<td><tt>0.13</tt></td>
<td><tt>22</tt></td>
<td><tt>0</tt></td>
<td><tt>162</tt></td>
</tr>

<tr valign="top">
<td><tt>agagtca</tt></td>
<td><tt>agagtca|tgactct</tt></td>
<td><tt>0.0001389909597</tt></td>
<td><tt>38</tt></td>
<td><tt>19.29</tt></td>
<td><tt>0.00011</tt></td>
<td><tt>8.9e-01</tt></td>
<td><tt>0.05</tt></td>
<td><tt>23</tt></td>
<td><tt>0</tt></td>
<td><tt>228</tt></td>
</tr>

<tr valign="top">
<td><tt>cccgaga</tt></td>
<td><tt>cccgaga|tctcggg</tt></td>
<td><tt>0.0000378751750</tt></td>
<td><tt>16</tt></td>
<td><tt>5.26</tt></td>
<td><tt>0.00012</tt></td>
<td><tt>1.0e+00</tt></td>
<td><tt>0.00</tt></td>
<td><tt>24</tt></td>
<td><tt>0</tt></td>
<td><tt>96</tt></td>
</tr>

</tbody><tfoot></tfoot></table><pre> Host name	pedagogix
 Job started	2020-01-31.101331
 Job done	2020-01-31.101334
 Seconds	2.47
	user	2.47
	system	0.07
	cuser	0.51
	csystem	0.03
 oligo-analysis  -v 1 -sort -i $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged -format fasta -lth occ_sig 0 -uth rank 50 -return occ,proba,rank -2str -noov -quick_if_possible -seqtype dna -bg upstream-noorf -org Candida_glabrata -pseudo 0.01 -l 8 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_8nt.tab
 Citation: van Helden et al. (1998). J Mol Biol 281(5), 827-42. 
 Program version              	1.169
 Quick counting mode          
 Detection of over-represented words (right-tail test)
 Oligomer length              	8
 Input file                   	$RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged
 Input format                 	fasta
 Output file                  	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_8nt.tab
 Discard overlapping matches
 Counted on both strands
 	grouped by pairs of reverse complements
 Background model             	upstream-noorf
 Organism                     	Candida_glabrata
 Background estimation method 	Frequency file
 Expected frequency file      	$RSAT/public_html/data/genomes/Candida_glabrata/oligo-frequencies/8nt_upstream-noorf_Candida_glabrata-noov-2str.freq
 Pseudo-frequency             	0.01
 Pseudo-frequency per oligo   	3.03988326848249e-07
 Sequence type                	DNA
 Nb of sequences              	141
 Sum of sequence lengths      	141000
 discarded residues           	NA (quick mode)	 (other letters than ACGT)
 discarded occurrences        	NA (quick mode)	 (contain discarded residues)
 nb possible positions        	NA (quick mode)
 total oligo occurrences      	138675
 total overlapping occurrences	916
 total non overlapping occ    	137759
 alphabet size                	4
 nb possible oligomers        	32896
 oligomers tested for significance	32888

 column headers
	1	seq            	oligomer sequence
	2	id             	oligomer identifier
	3	exp_freq       	expected relative frequency
	4	occ            	observed occurrences
	5	exp_occ        	expected occurrences
	6	occ_P          	occurrence probability (binomial)
	7	occ_E          	E-value for occurrences (binomial)
	8	occ_sig        	occurrence significance (binomial)
	9	rank           	rank
	10	ovl_occ        	number of overlapping occurrences (discarded from the count)
	11	forbocc        	forbidden positions (to avoid self-overlap)
</pre>

<table class="sortable" cellspacing="1" cellpadding="3" border="0"><thead><tr valign="top">
<th align="left"><tt>seq</tt></th>
<th align="left"><tt>id</tt></th>
<th align="left"><tt>exp_freq</tt></th>
<th align="left"><tt>occ</tt></th>
<th align="left"><tt>exp_occ</tt></th>
<th align="left"><tt>occ_P</tt></th>
<th align="left"><tt>occ_E</tt></th>
<th align="left"><tt>occ_sig</tt></th>
<th align="left"><tt>rank</tt></th>
<th align="left"><tt>ovl_occ</tt></th>
<th align="left"><tt>forbocc</tt></th>
</tr></thead>
<tbody>

<tr valign="top">
<td><tt>aggggggg</tt></td>
<td><tt>aggggggg|ccccccct</tt></td>
<td><tt>0.0001104338797</tt></td>
<td><tt>41</tt></td>
<td><tt>15.31</tt></td>
<td><tt>4e-08</tt></td>
<td><tt>1.3e-03</tt></td>
<td><tt>2.88</tt></td>
<td><tt>1</tt></td>
<td><tt>0</tt></td>
<td><tt>287</tt></td>
</tr>

<tr valign="top">
<td><tt>ccttcttc</tt></td>
<td><tt>ccttcttc|gaagaagg</tt></td>
<td><tt>0.0000379633478</tt></td>
<td><tt>19</tt></td>
<td><tt>5.26</tt></td>
<td><tt>2.9e-06</tt></td>
<td><tt>9.6e-02</tt></td>
<td><tt>1.02</tt></td>
<td><tt>2</tt></td>
<td><tt>0</tt></td>
<td><tt>133</tt></td>
</tr>

<tr valign="top">
<td><tt>gagtcatc</tt></td>
<td><tt>gagtcatc|gatgactc</tt></td>
<td><tt>0.0000246718091</tt></td>
<td><tt>15</tt></td>
<td><tt>3.42</tt></td>
<td><tt>3.3e-06</tt></td>
<td><tt>1.1e-01</tt></td>
<td><tt>0.97</tt></td>
<td><tt>3</tt></td>
<td><tt>0</tt></td>
<td><tt>105</tt></td>
</tr>

<tr valign="top">
<td><tt>ggggggga</tt></td>
<td><tt>ggggggga|tccccccc</tt></td>
<td><tt>0.0001692964079</tt></td>
<td><tt>48</tt></td>
<td><tt>23.48</tt></td>
<td><tt>6e-06</tt></td>
<td><tt>2.0e-01</tt></td>
<td><tt>0.71</tt></td>
<td><tt>4</tt></td>
<td><tt>0</tt></td>
<td><tt>336</tt></td>
</tr>

<tr valign="top">
<td><tt>caccagca</tt></td>
<td><tt>caccagca|tgctggtg</tt></td>
<td><tt>0.0000268870656</tt></td>
<td><tt>15</tt></td>
<td><tt>3.73</tt></td>
<td><tt>8.9e-06</tt></td>
<td><tt>2.9e-01</tt></td>
<td><tt>0.53</tt></td>
<td><tt>5</tt></td>
<td><tt>0</tt></td>
<td><tt>105</tt></td>
</tr>

<tr valign="top">
<td><tt>aatgactc</tt></td>
<td><tt>aatgactc|gagtcatt</tt></td>
<td><tt>0.0000303681828</tt></td>
<td><tt>16</tt></td>
<td><tt>4.21</tt></td>
<td><tt>9.2e-06</tt></td>
<td><tt>3.0e-01</tt></td>
<td><tt>0.52</tt></td>
<td><tt>6</tt></td>
<td><tt>0</tt></td>
<td><tt>112</tt></td>
</tr>

<tr valign="top">
<td><tt>actggctg</tt></td>
<td><tt>actggctg|cagccagt</tt></td>
<td><tt>0.0000218236223</tt></td>
<td><tt>13</tt></td>
<td><tt>3.03</tt></td>
<td><tt>1.8e-05</tt></td>
<td><tt>5.8e-01</tt></td>
<td><tt>0.24</tt></td>
<td><tt>7</tt></td>
<td><tt>0</tt></td>
<td><tt>91</tt></td>
</tr>

<tr valign="top">
<td><tt>cagggggg</tt></td>
<td><tt>cagggggg|cccccctg</tt></td>
<td><tt>0.0000253047395</tt></td>
<td><tt>14</tt></td>
<td><tt>3.51</tt></td>
<td><tt>1.9e-05</tt></td>
<td><tt>6.3e-01</tt></td>
<td><tt>0.20</tt></td>
<td><tt>8</tt></td>
<td><tt>0</tt></td>
<td><tt>98</tt></td>
</tr>

<tr valign="top">
<td><tt>cgcaccac</tt></td>
<td><tt>cgcaccac|gtggtgcg</tt></td>
<td><tt>0.0000078991533</tt></td>
<td><tt>8</tt></td>
<td><tt>1.10</tt></td>
<td><tt>2e-05</tt></td>
<td><tt>6.4e-01</tt></td>
<td><tt>0.19</tt></td>
<td><tt>9</tt></td>
<td><tt>0</tt></td>
<td><tt>56</tt></td>
</tr>

<tr valign="top">
<td><tt>gcaccagc</tt></td>
<td><tt>gcaccagc|gctggtgc</tt></td>
<td><tt>0.0000161272486</tt></td>
<td><tt>11</tt></td>
<td><tt>2.24</tt></td>
<td><tt>2.3e-05</tt></td>
<td><tt>7.5e-01</tt></td>
<td><tt>0.12</tt></td>
<td><tt>10</tt></td>
<td><tt>0</tt></td>
<td><tt>77</tt></td>
</tr>

<tr valign="top">
<td><tt>ctcccgag</tt></td>
<td><tt>ctcccgag|ctcgggag</tt></td>
<td><tt>0.0000041015708</tt></td>
<td><tt>6</tt></td>
<td><tt>0.57</tt></td>
<td><tt>2.9e-05</tt></td>
<td><tt>9.5e-01</tt></td>
<td><tt>0.02</tt></td>
<td><tt>11</tt></td>
<td><tt>0</tt></td>
<td><tt>42</tt></td>
</tr>

<tr valign="top">
<td><tt>agagtcat</tt></td>
<td><tt>agagtcat|atgactct</tt></td>
<td><tt>0.0000449255823</tt></td>
<td><tt>19</tt></td>
<td><tt>6.23</tt></td>
<td><tt>2.9e-05</tt></td>
<td><tt>9.5e-01</tt></td>
<td><tt>0.02</tt></td>
<td><tt>12</tt></td>
<td><tt>0</tt></td>
<td><tt>133</tt></td>
</tr>

</tbody><tfoot></tfoot></table><pre> Host name	pedagogix
 Job started	2020-01-31.101334
 Job done	2020-01-31.101341
 Seconds	6.28
	user	6.29
	system	0.15
	cuser	0.56
;	csystem	0.06
</pre>
<h2>Pattern assembly</h2>
<pre>pattern-assembly command: $RSAT/perl-scripts/pattern-assembly -v 1 -subst 1 -toppat 50 -2str -max_asmb_nb 10 -i $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.tab -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.asmb<p>
</p></pre><blockquote class="warning">
<font color="#FFAA00"><b>Warning: </b>Number of patterns = 86.	Exceeds top number = 50.	Only the top 50 patterns will be assembled.</font>
</blockquote><br><hr size="3">
<pre>; pattern-assembly  -v 1 -subst 1 -toppat 50 -2str -max_asmb_nb 10 -i $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.tab -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.asmb
; Input file	/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.tab
; Output file	/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.asmb
; Input score column           	8
; Output score column          	0
; Strand-insensitive assembly
; Max flanking bases           	1
; Max substitutions            	1
; Min matches                  	0
; Min weight                   	0
; Max assembly size            	50
; Max assembly width           	0
; Top number of patterns       	50
; Number of input patterns     	86
; THE ASSEMBLY WAS RESTRICTED TO THE	50 TOP PATTERNS
;
;Number of patterns = 86.	Exceeds top number = 50.	Only the top 50 patterns will be assembled.

; assembly # 1	seed: aggggg	5 assembled patterns	length 10
;   alignt	   rev_cpl	score
gaaggg....	....cccttc	2.27
.aagggg...	...cccctt.	4.07
..aggggg..	..ccccct..	7.58
...gggggg.	.cccccc...	1.83
....ggggga	tccccc....	1.35
gaagggggga	tccccccttc	7.58	best consensus

; assembly # 2	seed: gagtca	2 assembled patterns	length 7
;alignt	rev_cpl	score
ggagtc.	.gactcc	2.24
.gagtca	tgactc.	5.48
ggagtca	tgactcc	5.48	best consensus

; assembly # 3	seed: ggggta	2 assembled patterns	length 7
;alignt	rev_cpl	score
cggggt.	.accccg	0.94
.ggggta	tacccc.	4.18
cggggta	taccccg	4.18	best consensus

; assembly # 4	seed: gaagaa	3 assembled patterns	length 8
; alignt	 rev_cpl	score
aagaag..	..cttctt	0.47
.agaaga.	.tcttct.	1.83
..gaagaa	ttcttc..	3.48
aagaagaa	ttcttctt	3.48	best consensus

; assembly # 5	seed: caccag	5 assembled patterns	length 10
;   alignt	   rev_cpl	score
cgcacc....	....ggtgcg	0.94
.gcacca...	...tggtgc.	2.55
..caccag..	..ctggtg..	3.47
...accagc.	.gctggt...	2.31
....ccagcc	ggctgg....	1.47
cgcaccagcc	ggctggtgcg	3.47	best consensus

; assembly # 6	seed: catccc	1 assembled patterns	length 6
;alignt	rev_cpl	score
catccc	gggatg	2.70
catccc	gggatg	2.70	best consensus

; assembly # 7	seed: aagatg	1 assembled patterns	length 6
;alignt	rev_cpl	score
aagatg	catctt	2.25
aagatg	catctt	2.25	best consensus

; assembly # 8	seed: gtggaa	11 assembled patterns	length 16
;         alignt	         rev_cpl	score
ggctgg..........	..........ccagcc	1.47
.gctggt.........	.........accagc.	2.31
..ctggtg........	........caccag..	3.47
...tggtgg.......	.......ccacca...	1.73
....ggtgga......	......tccacc....	1.01
.....gtggaa.....	.....ttccac.....	1.81
......tggaag....	....cttcca......	0.59
.......ggaaga...	...tcttcc.......	1.51
........gaagaa..	..ttcttc........	3.48
.........aagaag.	.cttctt.........	0.47
..........agaaga	tcttct..........	1.83
ggctggtggaagaaga	tcttcttccaccagcc	3.48	best consensus

; assembly # 9	seed: gaaccc	1 assembled patterns	length 6
;alignt	rev_cpl	score
gaaccc	gggttc	1.70
gaaccc	gggttc	1.70	best consensus

; assembly # 10	seed: caccac	8 assembled patterns	length 13
;      alignt	      rev_cpl	score
cgcacc.......	.......ggtgcg	0.94
.gcacca......	......tggtgc.	2.55
..caccac.....	.....gtggtg..	1.61
...accacc....	....ggtggt...	1.59
....ccacca...	...tggtgg....	1.73
.....caccag..	..ctggtg.....	3.47
......accagc.	.gctggt......	2.31
.......ccagcc	ggctgg.......	1.47
cgcaccaccagcc	ggctggtggtgcg	3.47	best consensus
; Maximal number of assemblies is reached (10). Remaining patterns: 22.
; Host name	pedagogix
; Job started	2020-01-31.101341
; Job done	2020-01-31.101348
; Seconds	7.36
;	user	7.36
;	system	0.04
;	cuser	0
;	csystem	0
</pre>
<hr><pre><b>Command to generate matrices (PSSM):</b> $RSAT/perl-scripts/matrix-from-patterns -v 1 -logo  -seq $RSAT/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta -format fasta -asmb $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.asmb -min_weight 5 -flanks 2 -max_asmb_nb 10 -uth Pval 0.00025 -bginput -markov 0 -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm
</pre>
<hr>
<pre><b>Matrix conversion command:</b> $RSAT/perl-scripts/convert-matrix -v 1  -i $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt -from tab -to tab  -return counts -return consensus -return logo -logo_format png  -logo_opt '-e'  -logo_opt '-M'  -logo_dir $RSAT/public_html/tmp/www-data/2020/01/31/logos -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_2020-01-31.101409_UoF9BW_converted.tab
</pre><h2>Matrices</h2><pre>; convert-matrix  -v 1 -i $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt -from tab -to tab -return counts -return consensus -return logo -logo_format png -logo_opt -e -logo_opt -M -logo_dir $RSAT/public_html/tmp/www-data/2020/01/31/logos -o $RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_2020-01-31.101409_UoF9BW_converted.tab
; Input files
;	input	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt
; Input format        	tab
; Output files
;	output	$RSAT/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_2020-01-31.101409_UoF9BW_converted.tab
; Output format       	tab
; pseudo-weight       	1
a	73	71	23	161	170	11	16	18	16	18	20	157	81	70
c	34	26	10	13	3	4	4	2	6	6	5	8	25	29
g	74	76	186	43	46	204	201	200	202	199	188	55	85	85
t	49	57	11	13	11	11	9	10	6	7	17	10	39	46
//
; consensus	rrGAAGGGGGGArr
; consensus.rc	YYTCCCCCCTTCYY
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_1_l.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_1_l.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(1).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(1).png" height="120"></a>
;
a	82	100	38	17	293	16	14	15	288	88	103
c	55	49	8	20	5	11	6	261	4	72	67
g	75	81	244	263	3	273	9	17	8	48	60
t	102	84	24	14	13	14	285	21	14	106	84
//
; consensus	wdGGAGTCAww
; consensus.rc	WWTGACTCCHW
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_2_l.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_2_l.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(2).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(2).png" height="120"></a>
;
a	51	53	9	9	2	8	4	11	151	46	58
c	35	32	126	7	2	4	7	7	5	37	34
g	31	47	21	147	167	158	161	5	9	44	35
t	59	44	20	13	5	6	4	153	11	49	49
//
; consensus	wdCGGGGTAdw
; consensus.rc	WHTACCCCGHW
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_3_l.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_3_l.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(3).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(3).png" height="120"></a>
;
a	190	182	392	404	52	507	505	60	500	493	216	207
c	105	113	35	25	21	9	7	18	10	7	95	80
g	106	143	45	38	436	4	9	427	7	17	136	125
t	131	94	60	65	23	12	11	27	15	15	85	120
//
; consensus	arAAGAAGAAra
; consensus.rc	TYTTCTTCTTYT
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_4_l.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_4_l.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(4).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(4).png" height="120"></a>
;
a	16	17	11	5	0	65	5	6	65	1	6	11	18	12
c	24	26	49	11	68	3	57	61	2	7	64	52	23	21
g	17	11	6	54	3	2	7	0	0	61	0	4	14	20
t	16	19	7	3	2	3	4	6	6	4	3	6	18	20
//
; consensus	cycGCACCAGCCcb
; consensus.rc	VGGGCTGGTGCGRG
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_5_l.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_5_l.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(5).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(5).png" height="120"></a>
;
a	27	20	0	81	0	0	0	0	26	24
c	25	26	81	0	0	81	81	81	22	20
g	15	7	0	0	0	0	0	0	6	10
t	14	28	0	0	81	0	0	0	27	27
//
; consensus	myCATCCChw
; consensus.rc	WDGGGATGRK
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_6_l.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_6_l.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(6).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(6).png" height="120"></a>
;
a	36	34	114	114	0	114	0	0	41	30
c	26	23	0	0	0	0	0	0	23	32
g	26	34	0	0	114	0	0	114	22	19
t	26	23	0	0	0	0	114	0	28	33
//
; consensus	arAAGATGah
; consensus.rc	DTCATCTTYT
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_7_l.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_7_l.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(7).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(7).png" height="120"></a>
;
a	3	4	2	1	2	1	1	1	2	1	1	7	13	1	10	11	1	12	6	3
c	2	2	1	0	10	0	0	1	0	0	0	3	0	1	0	0	0	0	0	4
g	4	0	9	10	1	1	10	8	1	10	12	2	0	9	1	2	12	1	6	4
t	4	7	1	2	0	11	2	3	10	2	0	1	0	2	2	0	0	0	1	2
//
; consensus	kwgGCTGgTGGaAgAAGArs
; consensus.rc	SYTCTTCTTCCACCAGCCWM
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_8_l.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_8_l.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(8).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(8).png" height="120"></a>
;
a	25	18	0	64	64	0	0	0	22	15
c	9	5	0	0	0	64	64	64	15	13
g	13	18	64	0	0	0	0	0	9	11
t	17	23	0	0	0	0	0	0	18	25
//
; consensus	wdGAACCCwt
; consensus.rc	AWGGGTTCHW
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_9_l.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_9_l.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(9).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(9).png" height="120"></a>
;
a	7	11	6	4	4	39	1	5	42	4	4	44	4	3	5	15	13
c	16	19	30	6	33	4	40	37	3	36	34	1	6	39	33	14	8
g	11	4	6	35	3	3	5	2	1	6	4	1	33	2	6	9	11
t	13	13	5	2	7	1	1	3	1	1	5	1	4	3	3	9	15
//
; consensus	yycGCACCACCAGCCmw
; consensus.rc	WKGGCTGGTGGTGCGRR
<a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_10_.png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt_10_.png" height="120"></a><a href="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(10).png"><img src="./iHKG_TypeII_RSAT _ oligo-analysis result_files/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.t(10).png" height="120"></a>
;
; Host name	pedagogix
; Job started	2020-01-31.101410
; Job done	2020-01-31.101419
; Seconds	0.64
;	user	0.64
;	system	0.24
;	cuser	7.51
;	csystem	0.84
</pre><h2>Result file(s)</h2><table class="resultlink">
<tbody><tr><th>Content</th><th>URL</th></tr><tr><td>Input sequence</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta</a></td></tr><tr><td>Purged sequence</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta.purged</a></td></tr><tr><td>6-mers</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_6nt.tab">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_6nt.tab</a></td></tr><tr><td>7-mers</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_7nt.tab">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_7nt.tab</a></td></tr><tr><td>8-mers</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_8nt.tab">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_8nt.tab</a></td></tr><tr><td>merged oligos</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.tab">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.tab</a></td></tr><tr><td>Assembly</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.asmb">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.asmb</a></td></tr><tr><td>Significance matrices</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_sig_matrices.tf">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_sig_matrices.tf</a></td></tr><tr><td>Rescaled significance matrices</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_sig_matrices_rescaled.tf">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_sig_matrices_rescaled.tf</a></td></tr><tr><td>Final matrices (transfac format)</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.tf">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.tf</a></td></tr><tr><td>Final matrices (tab format)</td><td><a href="http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt">http://pedagogix-tagc.univ-mrs.fr/rsat//tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.txt</a></td></tr></tbody></table><hr size="3"><table class="nextstep" cellspacing="0" cellpadding="10" border="0" nowrap="">
<tbody><tr>

</tr><tr valign="top" align="center">
    <th valign="BOTTOM" align="CENTER" colspan="6">
	Next step
    </th>

</tr>

<tr><td valign="bottom" align="center">
<form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/dna-pattern_form.cgi">
<input type="hidden" name="title" value="">
<input type="hidden" name="pattern_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.tab">
<input type="hidden" name="sequence_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta">
<input type="hidden" name="sequence_format" value="fasta">
<input type="submit" value="string-based pattern matching (dna-pattern)">
</form>
</td>

<td valign="bottom" align="center">
<form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/pattern-assembly_form.cgi">
<input type="hidden" name="local_pattern_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.tab">
<input type="hidden" name="subst" value="1">
<input type="hidden" name="maxfl" value="1">
<input type="hidden" name="sc" value="auto">
<input type="hidden" name="strand" value="insensitive">
<input type="submit" value="pattern assembly">
</form>
</td>
</tr>

<tr><td valign="bottom" align="center"><form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/matrix-scan_form.cgi"><input type="hidden" name="matrix_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.tf"><input type="hidden" name="matrix_format" value="transfac"><input type="hidden" name="sequence_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/tmp_sequence_2020-01-31.101327_fGMMEv.fasta"><input type="hidden" name="sequence_format" value="fasta"><input type="submit" value="matrix-based pattern matching (matrix-scan)"></form></td></tr><tr><td valign="bottom" align="center"><form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/convert-matrix_form.cgi"><input type="hidden" name="matrix_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH_pssm_count_matrices.tf"><input type="hidden" name="matrix_format" value="transfac"><input type="submit" value="matrix conversion"></form></td></tr><tr></tr>

<tr>
<td valign="bottom" align="center">
<form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/XYgraph_form.cgi">
<input type="hidden" name="title1" value="oligo-analysis result">
<input type="hidden" name="title2" value="">
<input type="hidden" name="XYgraph_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.tab">
<input type="hidden" name="xcol" value="5">
<input type="hidden" name="xleg1" value="expected occurrences">
<input type="hidden" name="ycol" value="4">
<input type="hidden" name="yleg1" value="observed occurrences">
<input type="submit" value="XY graph">
</form>
</td>

  <td>
<form method="POST" action="http://pedagogix-tagc.univ-mrs.fr/rsat/classfreq_form.cgi">
<input type="hidden" name="transferred_file" value="/data/rsat/public_html/tmp/www-data/2020/01/31/oligo-analysis_2020-01-31.101327_9fzQpH.tab">
<input type="submit" value="Frequency distribution">
 </form>
  </td>
</tr>


</tbody></table>
<hr size="3">

</div></div><div id="cboxOverlay" style="display: none;"></div><div id="colorbox" class="" role="dialog" tabindex="-1" style="display: none;"><div id="cboxWrapper"><div><div id="cboxTopLeft" style="float: left;"></div><div id="cboxTopCenter" style="float: left;"></div><div id="cboxTopRight" style="float: left;"></div></div><div style="clear: left;"><div id="cboxMiddleLeft" style="float: left;"></div><div id="cboxContent" style="float: left;"><div id="cboxTitle" style="float: left;"></div><div id="cboxCurrent" style="float: left;"></div><button type="button" id="cboxPrevious"></button><button type="button" id="cboxNext"></button><button type="button" id="cboxSlideshow"></button><div id="cboxLoadingOverlay" style="float: left;"></div><div id="cboxLoadingGraphic" style="float: left;"></div></div><div id="cboxMiddleRight" style="float: left;"></div></div><div style="clear: left;"><div id="cboxBottomLeft" style="float: left;"></div><div id="cboxBottomCenter" style="float: left;"></div><div id="cboxBottomRight" style="float: left;"></div></div></div><div style="position: absolute; width: 9999px; visibility: hidden; display: none; max-width: none;"></div></div></body></html>